---
title: "Host Turkey B-cell Transcriptomics During THEV Infection Highlights Upregulated Cell Death and Breakdown Pathways That May Mediate Immunosuppression"
output:
  pdf_document:
    keep_tex: true
  officedown::rdocx_document:
    reference_docx: style_word_output.docx
geometry: margin=1in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{placeins}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage{threeparttablex}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
  - \usepackage{caption}
  - \usepackage[labelformat=empty]{caption}
bibliography: trxptome_refs.bib
csl: asm.csl
editor_options:
  chunck_output_type: console
  markdown:
  wrap: sentence
---

\vspace{5mm}

Abraham Quaye${^\dagger}$^,a^, Brett E. Pickett^a^, Joel S. Griffitts^a^, Bradford K. Berges^a^, Brian D. Poole^a,\*^

\vspace{5mm}
  
^a^Department of Microbiology and Molecular Biology, Brigham Young University\
${^\dagger}$Primary Author\
^\*^Corresponding Author

\vspace{5mm}

**Corresponding Author Information**\
[brian_poole\@byu.edu](mailto:brian_poole@byu.edu)\
Department of Microbiology and Molecular Biology,\
4007 Life Sciences Building (LSB),\
Brigham Young University,\
Provo, Utah\

\newpage
  
```{r setup, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}
library(knitr)
library(flextable)
opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)

inline_hook <- function(x) {
  if(is.numeric(x)){
    formatted <- format(x, big.mark = ",")
  }else{formatted <- x}
  paste0("**", formatted, "**")
}
knit_hooks$set(inline = inline_hook)
```

## ABSTRACT
\newpage
## INTRODUCTION

Turkey hemorrhagic enteritis virus (THEV), belonging to the family _Adenoviridae_, genus _Siadenovirus_, infects turkeys, chickens, and pheasants [@Harrach2008; @Davison2003]. Infecting its hosts via the feco-oral route, THEV causes hemorrhagic enteritis (HE) in turkeys, a debilitating disease affecting predominantly 6-12-week-old turkey poults characterized by immunosuppression (IMS), depression, splenomegaly, intestinal lesions leading to bloody droppings, and up to 80% mortality [@Gross1967; @Beach2006; @Dhama2017; @Tykaowski2019]. The clinical disease usually persists in affected flocks for about 7-10 days. However, secondary bacterial infections may extend the duration of illness and mortality for an additional 2-3 weeks due to the immunosuppressive nature of the virus, exacerbating the economic losses [@Pierson2008; @Dhama2017]. Low pathogenic (avirulent) strains of THEV have been isolated, which show subclinical infections but retain the immunosuppressive effects. Since its isolation from a pheasant spleen, the Virginia Avirulent Strain (VAS) has been used effectively as a live vaccine despite the immunosuppressive side-effects, but the vaccinated birds are rendered more susceptible to opportunistic infections and death than unvaccinated cohorts leading to significant economic losses [@Beach2006; @Rautenschlein2000; @Larsen1985; @Dhama2017; @Beach2009a]. 

It is well-established that THEV primarily infects and replicates in turkey B-cells of the bursa and spleen and somewhat in macrophages, inducing apoptosis and necrosis. Consequently, a significant drop in number of B-cells (specifically, IgM+ B-cells) and macrophages ensue along with increased T-cell counts with abnormal T-cell subpopulation (CD4+ and CD8+) ratios. The cell death seen in the B-cells and macrophages is generally proposed as the major cause of THEV-induced IMS as both humoral and cell-mediated immunity are impaired [@Tykaowski2019; @Rautenschlein2000; @Dhama2017; @Rautenschlein2000b]. It is also thought that the virus replication in the spleen attracts T-cells and peripheral blood macrophages to the spleen where the T-cells are activated by cytokines from activated macrophages and vice versa. The activated T-cells undergo clonal expansion and secrete interferons: type I (IFN-\(\alpha\) and IFN-\(\beta\)) and type II (IFN-\(\gamma\)) as well as tumor necrosis factor (TNF) while activated macrophages secrete interleukin 6 (IL-6), TNF, and nitric oxide (NO), an antiviral agent with immunosuppressive properties. The inflammatory cytokines released by T-cells and macrophages (e.g., TNF and IL-6) may also induce apoptosis in bystander splenocytes, exacerbating the already numerous apoptotic and necrotic splenocytes, culminating in IMS [@Rautenschlein2000; @Rautenschlein2000b] (see **Figure 1**). However, the precise molecular mechanisms of THEV-induced IMS or pathways involved are poorly understood [@Tykaowski2019]. Elucidating the specific mechanisms and pathways of THEV-induced IMS is the most crucial step in THEV research as it will present a means of mitigating the IMS.

Next generation sequencing (NGS) is a groundbreaking technology that has significantly enhanced our understanding of DNA and RNA structure and function and facilitated exceptional advancements in all domains of biology and the Life Sciences [@Satam2023]. mRNA sequencing (RNA-seq), an NGS approach to transcriptomic studies, is a versatile, high throughput, and cost-effective technology that allows a broad scan of the entire transcriptome, thereby uncovering the active genes and molecular pathways and processes. This technology has been leveraged in an ever increasing number of studies to elucidate active cellular processes under a wide range of treatment conditions, including the transcriptomics of viral infections [@Pandey2023; @Wang2019; @Choi2016; @Satam2023; @Mo2023]. In RNA-seq studies, differentially expressed genes (DEGs) identified under different experimental conditions are key to unlocking the interesting biology or mechanism under study. Identified DEGs are typically used for functional enrichment analysis in large curated knowledgebases which connect genes to specific biological processes, functions, and pathways such as gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways, shedding light on the biological question under study [@Ashburner2000; @Kanehisa2000].

To the best of our knowledge, no study has leveraged the wealth of information offered by RNA-seq to elucidate the molecular mechanisms and pathways leading to THEV-induced IMS. To effectively counteract the immunosupressive effect of the vaccine, it is essential to unravel the host cell mechanisms/pathways influenced by the virus to bring about IMS. In this study, we present the first transcriptomic profile of THEV-infected cells using paired-end RNA-seq in a turkey B-cell line (MDTC-RP19), highlighting key host genes, cellular/molecular processes and pathways affected during a THEV infection. Our RNA-seq yielded 149 bp long high quality (mean PHRED Score of 36) sequences from each end of cDNA fragments, which were mapped to the genome of domestic turkey (*Meleagris gallopavo*).
\newpage

## RESULTS
```{r mapping_stats}
source("scripts/r_code/reads_mapping_stats.R")

tot_raw_rds <- mapStats %>% pull(raw_reads) %>% sum(.)
tot_trm_rds <- mapStats %>% pull(trimmed_reads) %>% sum(.)
min_tr_rds <- mapStats %>% pull(trimmed_reads) %>% min(.)
max_tr_rds <- mapStats %>% pull(trimmed_reads) %>% max(.)

min_map_perc <- mapStats %>%
  mutate(map_perc = str_replace(mapped_reads, "\\d+\\.?\\d+\\s\\((\\d+\\.?\\d+)%\\)", "\\1")) %>%
  mutate(map_per = as.numeric(map_perc)) %>% pull(map_perc) %>%
  min(.) %>% as.numeric(.) %>% round(., 1)
max_map_perc <- mapStats %>%
  mutate(map_perc = str_replace(mapped_reads, "\\d+\\.?\\d+\\s\\((\\d+\\.?\\d+)%\\)", "\\1"),
         map_per = as.numeric(map_perc)) %>% pull(map_perc) %>%
  max(.) %>% as.numeric(.) %>% round(., 1)
```

**Sequencing Results**    
To identify the host transcriptome profile during THEV infection, MDTC-RP19 cells were THEV-infected or mock-infected in triplicates or duplicates, respectively, and collected in like manner at 4-, 12-, 24-, and 72-hours post infection (hpi). mRNAs extracted from mock- or THEV-infected cells were sequenced on the Illumina platform, yielding a total of `r tot_raw_rds` million raw reads (149 bp in length) across all samples (statistics for the sequencing reads obtained from each RNA library are presented in **Table 1**). After trimming off low-quality reads, the remaining `r tot_trm_rds` million total paired-end trimmed reads (approximately, `r min_tr_rds`-`r max_tr_rds` million reads per sample) were mapped to the genome of *Meleagris gallopavo* obtained from the National Center for Biotechnology Information (NCBI). The percentage of reads mapping to the host genome across all samples ranged from `r min_map_perc`-`r max_map_perc`%. Although our sequencing reads have excellent quality scores (see **Table 1**) at all time points, the DEGs identified at 4- and 72-hpi did not yield any results in the functional enrichment analyses (i.e, GO term and KEGG pathway analysis); hence, they were excluded from all subsequent analyses. In the remaining samples from 12- and 24-hpi, there is a high correlation was seen between biological replicates (**Figure 2A** and **B**)

**DEGs of THEV-infected Versus Mock-infected Cells**   
```{r DEG_stats, message=F}
source("scripts/r_code/extract_myDEG_geneIDs.R")

# extracting counts for number of genes
tot_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>% nrow(.) %>% format(., big.mark = ",")
tot_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>% nrow(.) %>% format(., big.mark = ",")

up_12 <- all_deg_tables[["up_degIDs_12hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
down_12 <- all_deg_tables[["down_degIDs_12hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
up_24 <- all_deg_tables[["up_degIDs_24hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
down_24 <- all_deg_tables[["down_degIDs_24hrs"]] %>% nrow(.) %>% format(., big.mark = ",")

# extracting log2FC ranges
min_fc_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  pull(log2fc) %>% min(.) %>% round(., 1)
min_fc_12_name <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  slice_min(log2fc) %>% pull(gene_name)
max_fc_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  pull(log2fc) %>% max(.) %>% round(., 1)
max_fc_12_name <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  slice_max(log2fc) %>% pull(gene_name)
min_fc_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  pull(log2fc) %>% min(.) %>% round(., 1) %>% format(., nsmall = 1)
min_fc_24_name <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  slice_min(log2fc) %>% pull(gene_name)
max_fc_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  pull(log2fc) %>% max(.) %>% round(., 1)
max_fc_24_name <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  slice_max(log2fc) %>% pull(gene_name)
```
Gene expression levels were estimated with the StringTie software [@Pertea2016] in Fragments per kilobase of transcript per million (FPKM) units. The analysis of DEGs was performed with the DESeq2 R package [@Deseq2] which employs negative binomial distribution model for read count comparisons. Using a P~adjusted~-value cutoff $\leq$ 0.05 as the inclusion criteria, a total of `r tot_12` and `r tot_24` genes were identified as differentially expressed at 12-hpi and 24-hpi, respectively. ~~DEG analyses results at 12 and 24-hpi are presented in **Supplementary Tables 1 and 2**, respectively~~. At 12-hpi, `r up_12` genes were upregulated and `r down_12` genes downregulated, whereas `r up_24` genes were upregulated and `r down_24` genes downregulated at 24-hpi (**Figure 2C**,and **Figure 3A-C**). The log~2~fold-change(FC) values at 12-hpi ranged between `r min_fc_12` and +`r max_fc_12` for `r min_fc_12_name` (Transmembrane Protein 156) and `r max_fc_12_name` (Lipase G), respectively. At 24-hpi, the log~2~FC values ranged between `r min_fc_24` and +`r max_fc_24` for `r min_fc_24_name` (C1q And TNF Related 12) and `r max_fc_24_name` (Potassium Voltage-Gated Channel Modifier Subfamily G Member 1), respectively.  
 
**Functional Enrichment Analyses (GO, KEGG pathway, and interaction network analyses)**  
Gene ontology (GO) enrichment analysis was performed for 12- and 24-hpi DEGs with the DAVID (Database for Annotation, Visualization and Integrated Discovery; version 2021) online resource [@Sherman2022] and the gprofiler2 R package -- version `r packageVersion("gprofiler2")` [@Gprofiler2], which output results in three GO categories -- cellular components (CP), biological processes (BP), and molecular functions (MF). Results with P~adjusted~-value $\leq$ 0.05 were considered functionally enriched. The GO enrichment analyses results at 12-hpi and 24-hpi showed significant intersections among all three GO categories. At both time points, cellular breakdown processes were upregulated while cellular maintenance processes and structures were downregulated in all three GO categories (**Table 2A-B** and **Table 3A-B**).

For upregulated DEGs at 12-hpi, GO terms annotated under the BP category broadly cluster into: apoptosis and autophagy, cellular metabolism (catabolic processes), sterol biosynthesis, response to stimuli, and protein processing (**Figure 4A** and **Table 2A**). Under the CC category, the GO terms relate primarily with cytoplasmic vacuolation while the GO terms under the MF category broadly fit under protein binding and kinase activity (**Table 2A**). For 12-hpi downregulated DEGs, the GO terms in BP category generally fall under: translation, protein biosynthesis and folding, ribosome biogenesis, nitrogen compound metabolism, nucleic acid synthesis, repair, metabolism, processing, and replication;  and energy metabolism. Also, immunoglobulin production and isotype switching was downregulated (**Figure 4C** and **Table 2B**). As for the CC category GO terms, they broadly group into: ribosome, mitochondria, respirosome, nucleus, and spliceosome while the MF category GO terms generally belong to: translation regulator activity, protein folding chaperone, catalytic activity (acting on a nucleic acids), and ATP hydrolysis activity (**Table 2B**).

At 24-hpi, the GO terms under the BP GO category for the upregulated DEGs are connected with: apoptosis and autophagy, lipid and sterol biosynthesis, catabolic process, protein ubiquitination and proteolysis, cell signalling, and cell metabolism. Additionally, host defense response and genes that negatively regulate cytokine production were upregulated (**Figure 4B** and **Table 3A**). The GO terms of the CC category, similar to those identified at 12-hpi, are also related with cytoplasmic vacuolation and the lysosome. The MF category GO terms group into: protein ubiquitination activity, kinase and acyltransferase activity, and macromolecule binding activity (**Table 3A**). The GO terms for the downregulated DEGs are markedly similar to those at 12-hpi in all three GO categories. The BP category GO terms broadly group into: translation, peptide biosynthesis and folding, ribosome biogenesis, aerobic respiration and ATP synthesis, and cell cycle process and nucleic acid replication and processing (**Figure 4D** and **Table 3B**). The GO terms of the CC category group under: ribosome, organelle, mitochondrion, nucleus and chromosomes while the MF category GO terms group into: structural constituent of ribosome and translation regulator activity, catalytic activity acting on a nucleic acid and nucleic acid binding, aminoacyl-tRNA ligase activity, and NAD binding (**Table 3B**).

KEGG analysis on the DEGs was performed using both the gprofiler2 R package [@Gprofiler2] and the DAVID online resource. Both analysis resources gave similar results but the results from DAVID (**Table 4A**) includes more information than the gprofiler2 results (**Table 4B**). The KEGG pathway analysis was congruent with the GO results, revealing that generally, cell maintenance and upkeep pathways were downregulated while cell death and breakdown pathways were upregulated. Cell maintenance pathways such as DNA replication and repair, ribosome biogenesis, spliceosome, and oxidative phosphorylation were downregulated at 12-hpi. Similar pathways were downregulated at 24-hpi. Pathways such as: autophagy, response to virus (Influenza A), and steroid biosynthesis were upregulated at 12-hpi similar to 24-hpi, where pathways such as: autophagy, ubiquitin-mediated proteolysis, lysosome, protein processing in endoplasmic reticulum, and steroid biosynthesis were upregulated.

It is well-established that THEV induces cell death (apoptosis and necrosis) in infected B-cells, which is linked to THEV-induced IMS [@Rautenschlein2000b; @Rautenschlein2000; @Saunders1993]. Hence, we are particularly interested in cellular processes and pathways associated with cell death and pathways that may affect the survivability of the host cells (B-cells), thereby accounting for THEV-induced IMS. We highlight the upregulated cell death (apoptosis and autophagy) pathways and responses to stimuli (especially ER-stress response) pathways identified by our GO and KEGG analyses as the likely key aspects of THEV-host cell interaction leading to IMS. 

**Cell Death and Breakdown Pathways Upregulated by THEV**   
Many virus families, including adenoviruses, herpesviruses, poxviruses, baculoviruses, parvoviruses, retroviruses, rhabdoviruses, pararmyxoviruses, orthomyxoviruses, togaviruses, and picornaviruses are known to trigger apoptosis in infected host cells either by a direct action of a viral protein or due to the host antiviral response [@Barber2001; @Hardwick1997; @Verburg2022]. The Mastadenovirus family possess the protein, E1B-19K, used to inhibit host cell apoptosis long enough to complete their replication cycle [@Ezoe1981; @Barber2001; @Verburg2022]. However, no such protein is known in THEV. A recent paper showed several novel transcripts and open reading frames (ORFs) in the genome of THEV which may offer similar anti-apoptotic functions but the functions of these novel ORFs are yet to be studied [@Quaye2024]. Our data show that apoptotic and autophagic pathways are upregulated during THEV infection, supporting previous findings of apoptosis and necrosis of THEV-infected cells [@Rautenschlein2000b; @Rautenschlein2000; @Saunders1993]. For example, several proapoptotic members of the BCL2 (B-cell lymphoma 2) protein family such as: BCL2 antagonist/killer 1 (BAK1), BCL2 interacting protein 3 like (BNIP3L), BCL2 interacting protein 3 (BNIP3), and Bcl2 modifying factor (BMF) were upregulated. Additionally, Fas cell surface death receptor (FAS), Fas associated via death domain (FADD), MAP kinase-activating death domain (MADD), programmed cell death 4(PDCD4), RB1 inducible coiled-coil 1 (RB1CC1), activating transcription factor 4 (ATF4), receptor interacting serine/threonine kinase 1 (RIPK1), tumor necrosis factor receptor superfamily member 1B (TNFRSF1B), pro-apoptotic WT1 regulator (PAWR), and apoptotic peptidase activating factor 1 (APAF1), which are potent proapoptotic factors were upregulated. Interestingly, both the intrinsic (BAK1, BNIP3L, BNIP3, BMF, RB1CC1, ATF4, PDCD4, and APAF1) and extrinsic (FAS, FADD, TNFRSF1B, MADD, and RIPK1) apoptotic pathways were represented. Conversely, we noted that, several anti-apoptotic proteins such as: BCL2 apoptosis regulator (BCL2), BCL2 interacting protein 2 (BNIP2; interacts directly with adenovirus E1B-19K protein), BCL2 related protein A1 (BCL2A1), and apoptosis inhibitor 5 (API5) were also upregulated. Thus, apoptosis and its regulation pathways are clearly upregulated; this likely highlights the host-virus tug-of-war and underscores the ability of adenoviruses to trigger both apoptotic and anti-apoptotic pathways as seen in Mastadenoviruses. Moreover, several genes associated with autophagy such as: TNF receptor associated factor 6 (TRAF6), autophagy related 9A (ATG9A), unc-51 like autophagy activating kinase 2 (ULK2), and autophagy related 4B cysteine peptidase (ATG4B) were upregulated.

We also see several essential cell maintenance processes whose downregulation can trigger apoptosis. For instance, severe DNA damage is a known mechanism of apoptosis induction, which is called DNA damage-dependent apoptosis [@Roos2006]. Repression of RNA and protein synthesis are also strongly associated with apoptosis [@Martin1993]. Several processes related nucleic acid (DNA and RNA) synthesis, maintenance, and repair such as: nucleotide biosynthesis and metabolism, double strand break repair, DNA excision repair, RNA biosynthesis, RNA processing, DNA replication, mitotic cell cycle process, 	protein-RNA complex organization, and DNA damage response were downregulated. Some notable genes identified include: DNA ligase 1 (LIG1), X-ray repair cross complementing 1 (XRCC1), cyclin dependent kinase 1 and 2 (CDK1, CDK2), checkpoint kinase 1 (CHEK1), 8-oxoguanine DNA glycosylase (OGG1), BLM RecQ-like-helicase (BLM), BRCA1 DNA repair associated (BRCA1), and several RAD family proteins (RAD21, RAD51, RAD51B, RAD51C, RAD54B). Additionally, protein-related  processes including: ribosome biogenesis, rRNA processing, ribosome assembly, protein folding, translational initiation, protein maturation, ribosome and ribonucleoprotein complex formation, translation pre-initiation complex formation, and cytoplasmic translation were significantly downregulated. We identified notable genes such as: eukaryotic translation initiation factors -- EIF1, EIF1AX, EIF3E and EIF3F, EIF3H, EIF3I, EIF3L and EIF3M -- biogenesis of ribosomes BRX1 (BRIX1), MCTS1 re-initiation and release factor (MCTS1), and ribosomal protein subunits (L10a, L11, 12, 13, 14, 15, 18a, 19).

**Cellular Responses to Stimuli during THEV infection**   
The transcriptome profile indicates that THEV infection likely induced notable ER stress, oxidative stress, and biotic stimuli. As an important cellular organelle, the endoplasmic reticulum (ER) is the major site for protein synthesis, folding and sorting. The unfolded protein response (UPR) would be activated by ER homeostasis disturbance such as ER stress [###]. In our results, following THEV infection, X box binding protein 1 (Xbp-1), a marker of inositol-requiring protein 1 (IRE1) branch upon UPR induction, was upregulated. Another protein involved in ER, HSPA5 [Heat Shock Protein Family A (Hsp70) Member 5], was induced almost 8.4 folds in activating transcriptional factor 6 (ATF6) branch of UPR. Besides, other ER-stress involved genes such as homocysteine-inducible endoplasmic reticulum stress-inducible ubiquitin-like domain member 1 (HERPUD1), DNA damage inducible transcript 3 (DDIT3), and cation transport regulator-like protein 1 (CHAC1) that is a novel gene identified in activating transcription factor 4 (ATF4) arm of the PKR-like ER protein kinase (PERK) branch of the UPR were upregulated ranking at the top 3 with the induction folds of 12.5, 10.2 and 11.9, respectively (Supplementary Table S1 and Fig. 6B). These data suggest that upon CCHFV
infection, three UPRs pathways mediated by IRE1, ATF6 and PERK were likely all stimulated

**Cellular Metabolism Changes During THEV Infection**   
Many viruses, such as hepatitis C virus, human cytomegalovirus, influenza virus, and rhinovirus, have been documented to manipulate cellular metabolism processes to their advantage [@Mo2023].
\newpage

## DISCUSSION
We may not have seen a measurable immune response/pathway enrichment in the infected host cells because the these B-cells may likely require other immune cells such as macrophages and T-cells for activation/mount an immune response. Also, those cytokine measurement were recorded in cell culture or splenocytes (not just B cells). Additionally, cytokines may not play a dominant role in THEV-induced IMS since TNF-blocking drug (thalidomide) only prevented intestinal disease not immunosuppression, suggesting that the mechanism of IMS and intestinal disease are distinct. Secondly, the curated data in the GO, and KEGG databases are most complete for human and other model organisms; hence, there may not be enough information curated for turkeys to highlight the anything that is not very strong immune response.
\newpage

## CONCLUSIONS
\newpage

## MATERIALS AND METHODS

**Cell culture and THEV Infection**   
The Turkey B-cell line (MDTC-RP19, ATCC CRL-8135) was grown as a suspension culture in 1:1 complete Leibovitz's L-15/McCoy's 5A medium with 10% fetal bovine serum (FBS), 20% chicken serum (ChS), 5% tryptose phosphate broth (TPB), and 1% antibiotic solution (100 U/mL Penicillin and 100$\mu g$/mL Streptomycin), at 41^o^C in a humidified atmosphere with 5% CO~2~. Infected cells were maintained in 1:1 serum-reduced Leibovitz's L15/McCoy's 5A media (SRLM) with 2.5% FBS, 5% ChS, 1.2% TPB, and 1% antibiotic solution. A commercially available THEV vaccine was purchased from Hygieia Biological Labs (VAS strain). The stock virus was titrated using an in-house qPCR assay with titer expressed as genome copy number (GCN)/mL, similar to Mahshoub *et al* [@Mahsoub2017]. Cells were THEV-infected or mock-infected in triplicates or duplicates, respectively at a multiplicity of infection (MOI) of 100 GCN/cell, incubated at 41^o^C for 1 hour, and washed three times with phosphate buffered saline (PBS) to get rid of free virus particles. At each time point (4-, 12-, 24-, and 72-hpi), triplicate (THEV-infected) and duplicate (mock-infected) samples were harvested for total RNA extraction.

**RNA extraction and Sequencing**   
Total RNA was extracted from infected cells using the Thermofisher RNAqueous™-4PCR Total RNA Isolation Kit (which includes a DNase I digestion step) per manufacturer's instructions. An agarose gel electrophoresis was performed to check RNA integrity. The RNA quantity and purity was initially assessed using nanodrop, and RNA was used only if the A260/A280 ratio was 2.0 ± 0.05 and the A260/A230 ratio was \>2 and \<2.2. Extracted total RNA samples were sent to LC Sciences, Houston TX for poly-A-tailed mRNA sequencing. RNA integrity was checked with Agilent Technologies 2100 Bioanalyzer High Sensitivity DNA Chip and poly(A) RNA-seq library was prepared following Illumina's TruSeq-stranded-mRNA sample preparation protocol. Paired-end sequencing, generating 150 bp reads was performed on the Illumina NovaSeq 6000 sequencing system. The paired-end 150bp sequences obtained during this study and all expression data have been submitted to the Gene Expression Omnibus database, under accession no #######

**Quality Control and Mapping Process**   
Sequencing reads were processed following a well-established protocol described by Pertea *et al* [@Pertea2016], using Snakemake - version 7.32.4 [@Snakemake2021], a popular workflow management system to drive the pipeline. Briefly, raw sequencing reads were trimmed with Cutadapt - version 1.10 [@Martin2011] and the quality of trimmed reads evaluated using the FastQC software, version 0.12.1 (Bioinformatics Group at the Babraham Institute, Cambridge, United Kingdom; www.bioinformatics.babraham.ac.uk), achieving an overall Mean Sequence Quality (PHRED Score) of 36. Trimmed reads were mapped the reference *Meleagris gallopavo* genome (<https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/146/605/GCF_000146605.3_Turkey_5.1/GCF_000146605.3_Turkey_5.1_genomic.fna.gz>) with Hisat2 - version 2.2.1 [@Pertea2016] using the accompanying gene transfer format (GTF) annotation file (<https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/146/605/GCF_000146605.3_Turkey_5.1/GCF_000146605.3_Turkey_5.1_genomic.gtf.gz>) to build a genomic index. Samtools - version 1.19.2 was used to convert the output Sequence Alignment Map (SAM) file to the more manageable Binary Alignment Map (BAM) format. The StringTie (v2.2.1) software [@Pertea2016], set to expression estimation mode was used to generate normalized gene expression estimates from the BAM files for genes in the reference GTF file after which the prepDE.py3 script was used to extract read count information from the StringTie gene expression files, providing an expression-count matrix for downstream DEG analysis.

**DEG Analysis and Functional Enrichment Analysis**   
DEG analysis between mock- and THEV-infected samples was performed using the very popular DESeq2 [@Deseq2], which employs a Negative Binomial distribution model for read count comparisons. Genes with P~adjusted~-value $\leq$ 0.05 were considered as differentially expressed. The read count data are deposited at Gene Expression Omnibus (GEO) under accession number ###. The functional profiling of DEGs (GO and KEGG analyses) were performed based on GO databases and KEGG databases using the R package gprofiler2 [@Gprofiler2] with *Meleagris gallopavo* as the reference organism. Results with P~adjusted~-value $\leq$ 0.05 were included as functionally enriched. Additionally, the DAVID (Database for Annotation, Visualization and Integrated Discovery; version 2021) online analysis tool was used for KEGG pathway analysis. All visualization plots were made using ggplot2, pheatmap, and ggvenn R packages [@ggplot2; @pheatmap; @ggvenn].

**Validation of DEGs by Reverse Transcriptase Quantitative PCR (RT-qPCR)**

**Statistical Analysis**
\newpage

## DATA AVAILABILITY
\newpage

## CODE AVAILABILITY
\newpage

## ACKNOWLEDGMENTS
\newpage

## REFERENCES

```{=tex}
\setlength{\parindent}{-0.25in}
\setlength{\leftskip}{0.25in}
```
\noindent

::: {#refs}
:::

```{=tex}
\setlength{\parindent}{0in}
\setlength{\leftskip}{0in}
\newpage
```
\newpage

## TABLES AND FIGURES

![**Figure 1: Model of THEV-induced immunosuppression in turkeys**. THEV infection of target cells is indicated with black dotted arrows. Black unbroken arrows indicate cell activation. Red arrows indicated signals leading to apoptosis. Blue arrows indicate all cytokines released by the cell. Blue arrows with square heads indicated an event leading to IMS. Adapted from Rautenschlein *et al.* [@Rautenschlein2000].](results/imss_model.png)

![**Figure 2. (A) Principal component analysis (PCA) of turkey B-cells during THEV infection.** At 12-hpi **(A1)**,the results indicate that the first (PC1) and second (PC2) principal components account for 96% and 3% of the variation in the samples, respectively. Whereas PC1 and PC2 account for 96% and 2% of the variation, respectively at 24-hpi **(A2)**. **(B) Poisson distance matrices illustrating the RNA-seq library integrity within treatment (infected versus mock) groups**. The color scale represents the distances between biological replicates for both 12-hpi samples **(B1)** and 24-hpi samples **(B2)**. Dark colors represent high correlation (similarity) between the samples involved. **(C) Volcano plots of DEGs between THEV-infected versus mock-infected cells at 12- and 24-hpi**. Red, blue, and grey dots represent upregulated, downregulated, and non-significant genes, respectively for both 12-hpi samples **(C1)** and 24-hpi samples **(C2)**.](results/r/figures/sample_corr_figure.png)

![**Figure 3: Differentially expressed genes (DEGs) of THEV-infected versus mock-infected samples at different time points. (A) Bar plot of number DEGs identified.** Red represents upregulated genes and blue represents downregulated genes.  **(B) Heatmaps of scaled expression data (Z-scores) of DEGs.** DEGs identified at 12-hpi are shown in **(B1)** and DEGs at 24-hpi in **(B2).** **(C) Venn diagrams showing the number of DEGs identified at different time points.** For the upregulated genes **(C1)**, the red circle represents genes at 4-hpi, the blue circle, 12-hpi, and the grey circle, 24-hpi. For the downregulated genes **(C2)**, the green circle represents genes at 72-hpi, while all the other time points retain the colors from **(C1)**.](results/r/figures/deg_patch_fig.png)

![**Figure 4: Dotplot of Enriched Gene Ontology Biological Processes (BP)**. Significant BP GO terms identified for upregulated DEGs at 12-hpi and 24-hpi are shown in **(A)** and **(B)**, respectively. Significant BP GO terms for downregulated DEGs at 12-hpi and 24-hpi are shown in **(C)** and **(D)**, respectively. The y-axis indicates GO terms and the x-axis represents the rich factor, which indicates the ratio of the number of DEGs annotated to the term to the total number of genes annotated to the term. The diameter indicates the number of genes overlapping the gene ontology term and the color indicates the enrichment P-value.](results/r/figures/patch_GO_enrich.png)

\newpage
```{r, mapping_stats_table}
set_flextable_defaults(font.size = 6, table.layout = "fixed", float = "none")
mapStats %>%
  flextable(.) %>%
  theme_vanilla() %>%
  set_header_labels(sample = "Sample", raw_reads = "Raw Reads", trimmed_reads = "Trimmed Reads",
                    mapped_reads = "Mapped Reads", unq_map_reads = "Uniquely Mapped \nReads",
                    multi_map_reads = "Non-uniquely \nMapped Reads", q20 = "Q20%",
                    q30 = "Q30%", gc_content = "GC\nContent (%)") %>%
  flextable::font(part = "all", fontname = "Times New Roman") %>%
  flextable::bold(part = "header") %>%
  flextable::set_table_properties(layout = "fixed", align = "center",
                                  opts_pdf = list(caption_repeat = F),
                                  opts_word = list(split = F, keep_with_next = T)) %>%
  flextable::align(align = "center", part = "all") %>%
  footnote(., j = c(2:6),
           value = as_paragraph(c("All values for number of reads are in millions")),
           ref_symbols = c("M"), part = "header", inline = T) %>%
  footnote(., j = 1, i = c(1:11),
           value = as_paragraph(c("These are infected samples indicated by the letter 'I' and 'S' in sample names")),
           ref_symbols = c("Inf"), part = "body", inline = F) %>%
  footnote(., j = 1, i = c(12:19),
           value = as_paragraph(c("These are mock-infected samples indicated by the letters 'U' and 'N' in sample names")),
           ref_symbols = c("Mk"), part = "body", inline = F) %>%
  flextable::height(part = "header", height = 0.6) %>%
  flextable::height(part = "body", height = 0.2) %>%
  set_caption("Table 1: Summary of sequencing, quality control, and mapping processes")

```

```{r GOTerm_Enrich_UP_12hpi}
source("scripts/r_code/generate_GO_tables.R")
# Table 2A
make_GO_table <- function(goTerms_res){
  
  # Calculate row number to merge GOTerm category label
  bp_row_num <-  goTerms_res %>%
    filter(source %in% c("GO:BP", "Biological Process")) %>% nrow(.) + 1
  cc_row_num <- goTerms_res %>%
    filter(source %in% c("GO:CC", "Cellular Component")) %>% nrow(.) + bp_row_num
    
  table <- goTerms_res %>% 
    flextable() %>%
    theme_vanilla() %>%
    set_header_labels(source = "GO Category", term = "GO:Term",
                      qval = "P-value\n(Adjusted)",
                      num_degs = "Number of DEGs",
                      fold_enrich = "Fold Enrichment") %>%
    flextable::font(part = "all", fontname = "Times New Roman") %>%
    flextable::bold(part = "header") %>%
    flextable::set_table_properties(layout = "fixed", align = "center",
                                    opts_pdf = list(caption_repeat = F),
                                    opts_word = list(split = F, keep_with_next = T)) %>%
    flextable::align(align = "center", part = "all") %>%
    merge_at(., i = 1, j = 1:5, part = "body") %>%
    merge_at(., i = bp_row_num, j = 1:5, part = "body") %>%
    merge_at(., i = cc_row_num, j = 1:5, part = "body") %>%
    flextable::bold(part = "body", i = c(1, bp_row_num, cc_row_num)) %>%
    flextable::fontsize(part = "body", size = 6) %>%
    flextable::fontsize(part = "body", i = c(1, bp_row_num, cc_row_num), size = 8) %>%
    flextable::width(., j = 2, width = 3.5) %>%
    flextable::height(., height = 0.03) %>%
    flextable::height(., height = 0.1, i = c(1, bp_row_num, cc_row_num)) %>%
    flextable::height(part = "header", height = 0.2)
  
  return(table)
}

go_res_table12up <- tab_res %>% filter(timepoint == "t12") %>% select(go_tab_up) %>% unnest(.)

make_GO_table(go_res_table12up) %>%
    set_caption("Table 2A: Gene ontology analysis of Significantly Upregulated DEGs identified at 12-hpi")

```

```{r GOTerm_Enrich_DOWN_12hpi}
# Table 2B
go_res_table12dw <- tab_res %>% filter(timepoint == "t12") %>% select(go_tab_down) %>% unnest(.)

make_GO_table(go_res_table12dw) %>%
    set_caption("Table 2B: Gene ontology analysis of Significantly Downregulated DEGs identified at 12-hpi")

```

```{r GOTerm_Enrich_UP_24hpi}
# Table 3A
go_res_table24up <- tab_res %>% filter(timepoint == "t24") %>% select(go_tab_up) %>% unnest(.)

make_GO_table(go_res_table24up) %>%
    set_caption("Table 3A: Gene ontology analysis of Significantly Upregulated DEGs identified at 24-hpi")

```

```{r GOTerm_Enrich_DOWN_24hpi}
# Table 3B
go_res_table24dw <- tab_res %>% filter(timepoint == "t24") %>% select(go_tab_down) %>% unnest(.)

make_GO_table(go_res_table24dw) %>%
    set_caption("Table 3B: Gene ontology analysis of Significantly Downregulated DEGs identified at 24-hpi")
```

```{r KEGG_result_DAVID}
# Load script with KEGG result tables
source("scripts/r_code/process_DAVID_kegg.R")

david_kegg <- kegg_result %>%
  mutate(across(c(num_degs:qval), as.numeric),
         qval = format(qval, digits = 3)) %>%
  select(-c(rich_factor, pval)) %>%
  mutate(regulation = str_replace(timepoint, "^((down|up))\\d{2}hrs$", "\\1"),
         timepoint = str_replace(timepoint, "^(down|up)(\\d{2})hrs$", "\\2-hpi")) %>%
  select(timepoint, regulation, everything())
  

david_kegg %>% flextable(.) %>%
  theme_vanilla() %>% 
  width(., j = "term", width = 2.5) %>%
  set_header_labels(timepoint = "Time Point", regulation = "Regulation",
                    term = "KEGG Term", num_degs = "DEG Count",
                    fold_enrich = "Fold Enrichment", qval = "P-value\n(Adjusted)") %>%
  flextable::align(align = "center", part = "all") %>%
  set_caption("Table 4A: Significantly Enriched KEGG Pathways from DEGs identified at 12 and 24-hpi (Results from the DAVID online resource)")

```

```{r KEGG_result_grofiler2}

gprofiler_kegg <- kegg_res_all %>%
  mutate(regulation = str_replace(query, "^(down|up)_degIDs_\\d{2}hrs$", "\\1"),
         timepoint = str_replace(query, "^(down|up)_degIDs_(\\d{2})hrs$", "\\2-hpi")) %>%
  select(timepoint, regulation, term_name, deg_count = intersection_size, p_value) %>%
  mutate(p_value = format(p_value, digits = 3))
  

gprofiler_kegg %>% flextable(.) %>%
  theme_vanilla() %>% 
  width(., j = "term_name", width = 3) %>%
  set_header_labels(timepoint = "Time Point", regulation = "Regulation",
                    term_name = "KEGG Term", deg_count = "DEG Count",
                      p_value = "P-value (Adjusted)") %>%
  flextable::align(align = "center", part = "all") %>%
  set_caption("Table 4B: Significantly Enriched KEGG Pathways from DEGs identified at 12 and 24-hpi (Results from the gprofiler2 R package)")

```

\newpage

## SUPPLEMENTARY INFORMATION/MATERIALS
