---
title: "Turkey B cell Transcriptome Profile During Turkey Hemorrhagic Enteritis Virus (THEV) Infection Highlights Upregulated Apoptosis and Breakdown Pathways That May Mediate Immunosuppression"
output:
  pdf_document:
    keep_tex: true
  officedown::rdocx_document:
    reference_docx: style_word_output.docx
geometry: margin=1in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{placeins}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage{threeparttablex}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
  - \usepackage{caption}
  - \usepackage[labelformat=empty]{caption}
bibliography: trxptome_refs.bib
csl: asm.csl
editor_options:
  chunck_output_type: console
  markdown:
  wrap: sentence
---

\vspace{5mm}

Abraham Quaye, Brett E. Pickett, Joel S. Griffitts, Bradford K. Berges, Brian D. Poole^\*^

\vspace{5mm}
  
Department of Microbiology and Molecular Biology, Brigham Young University\
^\*^Corresponding Author

\vspace{5mm}

**Corresponding Author Information**\
[brian_poole\@byu.edu](mailto:brian_poole@byu.edu)\
Department of Microbiology and Molecular Biology,\
4007 Life Sciences Building (LSB),\
Brigham Young University,\
Provo, Utah\

\newpage
  
```{r setup, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}
library(knitr)
library(flextable)
opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)

inline_hook <- function(x) {
  if(is.numeric(x)){
    formatted <- format(x, big.mark = ",")
  }else{formatted <- x}
  paste0("**", formatted, "**")
}
knit_hooks$set(inline = inline_hook)
```

## Abstract
Infection with Turkey Hemorrhagic Enteritis Virus (THEV) can cause hemorrhagic enteritis, which affects young turkeys. This disease is characterized by bloody diarrhea and immunosuppression (IMS), which is attributed to apoptosis of infected B cells. Secondary infections due to IMS exacerbates economic losses. We performed the first transcriptomic analysis of a THEV infection to elucidate the mechanisms mediating THEV-induced IMS. After infecting and sequencing mRNAs of a turkey B-cell line, trimmed reads were mapped to the host turkey genome and gene expression was quantified with StringTie. Differential gene expression analysis was followed by functional enrichment analyses using gprofiler2 and DAVID from NCBI. RT-qPCR of select genes was performed to validate the RNA-seq data. A total of 2,343 and 3,295 differentially expressed genes (DEGs) were identified at 12-hpi and 24-hpi, respectively. The DEGs correlated with multiple biological processes including apoptosis, ER unfolded protein response, and cell maintenance. Multiple pro-apoptotic genes, including *APAF1, BMF, BAK1, and FAS* were upregulated. Genes that play a role in ER stress-induced unfolded protein response including *VCP, UFD1, EDEM1, and ATF4* were also upregulated and may contribute to apoptosis. Our data suggest that several biological processes and pathways including apoptosis and ER response to stress are important aspects of the host cell response to THEV infection. It is possible that interplay between multiple processes may mediate apoptosis of infected B-cells, leading to IMS.

## Keywords
Turkey hemorrhagic enteritis virus (THEV), Adenovirus, RNA sequencing, Apoptosis, Immunosuppression, ER stress, B cell.
\newpage

## 1. Introduction     
Turkey hemorrhagic enteritis virus (THEV) belongs to genus _Siadenovirus_, family _Adenoviridae_, and infects turkeys, chickens, and pheasants [@Harrach2008; @Davison2003]. THEV is transmitted via the fecal-oral route and causes hemorrhagic enteritis (HE) in turkeys, a debilitating disease affecting predominantly 6-12-week-old turkey poults characterized by immunosuppression (IMS), lack of vitality, splenomegaly, intestinal lesions leading to bloody diarrhea, and up to 80% mortality [@Gross1967; @Beach2006; @Dhama2017; @Tykaowski2019]. The clinical disease usually persists in affected flocks for 7-10 days, causing death and economic losses. However, secondary bacterial infections may extend the duration of illness and increase mortality for an additional 2-3 weeks due to the immunosuppressive nature of the virus, exacerbating the economic losses [@Pierson2008; @Dhama2017]. Naturally occurring low pathogenic (avirulent) strains of THEV have been isolated; these strains cause subclinical infections but retain the immunosuppressive effects. Since its isolation from a pheasant spleen, the Virginia Avirulent Strain (VAS) has been effectively used as a live vaccine despite the immunosuppressive side-effects. However, the vaccinated birds are rendered more susceptible to opportunistic infections and death than unvaccinated cohorts [@Beach2006; @Rautenschlein2000; @Larsen1985; @Dhama2017; @Beach2009a]. 

It is well established that THEV primarily infects and replicates in turkey B-cells of the bursa and spleen and to a lesser extent in macrophages, inducing apoptosis and necrosis [@Rautenschlein2000; @Tykaowski2019]. Consequently, a significant drop in the number of B-cells (specifically, IgM+ B-cells) and macrophages ensues along with increased T-cell counts with abnormal ratios of T-cell subpopulations (CD4+ and CD8+) [@Rautenschlein2000b; @Rautenschlein2000; @Tykaowski2019]. The cell death seen in the infected B-cells and macrophages is generally proposed as the major cause of THEV-induced IMS as both humoral and cell-mediated immunity are impaired [@Rautenschlein2000; @Dhama2017]. Immunopathogenesis via cytokines from T-cells and macrophages has also been suggested as a mechanism of apoptosis leading to IMS. It is thought that virus replication in the spleen attracts T-cells and peripheral blood macrophages, which results in T-cell activation by cytokines from activated macrophages and vice versa. The activated T-cells undergo clonal expansion and secrete type I (IFN-\(\alpha\) and IFN-\(\beta\)) and type II (IFN-\(\gamma\)) interferons as well as tumor necrosis factor (TNF), while activated macrophages secrete interleukin 6 (IL-6), TNF, and nitric oxide (NO). These cytokines may further contribute to apoptosis and necrosis in bystander splenocytes, culminating in IMS [@Rautenschlein2000; @Rautenschlein2000b] (**Figure 1**). However, the precise molecular mechanisms of THEV-induced IMS or the relevant intracellular signaling pathways are poorly understood [@Tykaowski2019]. Elucidating the specific mechanisms and pathways of THEV-induced IMS is a crucial step in THEV research as it could present a means of mitigating IMS.

Bulk mRNA sequencing (RNA-seq), a next generation sequencing approach to transcriptomic studies, is a versatile, high throughput, and cost-effective technology that allows a broad survey of the entire transcriptome of a cell population, thereby uncovering the active genes and molecular pathways and processes. This technology has been leveraged in an ever-increasing number of studies to elucidate active cellular processes under a wide range of treatment conditions, including viral infections [@Pandey2023; @Wang2019; @Choi2016; @Satam2023; @Mo2023]. In RNA-seq studies, differentially expressed genes (DEGs) identified by contrasting pairs of different experimental conditions are key to unlocking the interesting biology or mechanism. Identified DEGs are typically used for functional enrichment analyses in large curated knowledgebases such as gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways which connect genes to specific biological processes, functions, and pathways, shedding light on the biological question under study [@Ashburner2000; @Kanehisa2000].

To our knowledge, no study has used RNA-seq to elucidate the molecular mechanisms and pathways leading to THEV-induced IMS. To effectively counteract the immunosuppressive effect of the vaccine, it is essential to unravel the host cell processes/pathways influenced by the virus to bring about IMS. In this study, we present the first transcriptomic profile of THEV-infected cells using paired-end bulk RNA-seq in a turkey B-cell line (MDTC-RP19), highlighting key host genes, cellular/molecular processes and pathways affected during a THEV time course infection. We specifically focus on cellular processes related to cell survivability that can elucidate THEV-induced IMS.
\newpage

## 2. Materials and  Methods       
### Cell culture and Infection    
The Turkey B-cell line (MDTC-RP19, ATCC CRL-8135) was grown as a suspension culture in 1:1 complete Leibovitz's L-15/McCoy's 5A medium with 10% fetal bovine serum (FBS), 20% chicken serum (ChS), 5% tryptose phosphate broth (TPB), and 1% antibiotic solution (100 U/mL Penicillin and 100$\mu g$/mL Streptomycin), at 41^o^C in a humidified atmosphere with 5% CO~2~. Infected cells were maintained in 1:1 serum-reduced Leibovitz's L15/McCoy's 5A media (SRLM) with 2.5% FBS, 5% ChS, 1.2% TPB, and 1% antibiotic solution. A commercially available THEV vaccine was purchased from Hygieia Biological Labs (VAS strain). The stock virus was titrated using an in-house qPCR assay with titer expressed as genome copy number (GCN)/mL, similar to Mahshoub *et al* [@Mahsoub2017]. Cells were THEV-infected or mock-infected in triplicates or duplicates, respectively at a multiplicity of infection (MOI) of 100 GCN/cell, incubated at 41^o^C for 1 hour, and washed three times with phosphate buffered saline (PBS) to remove unattached virus particles. At each time point (4-, 12-, 24-, and 72-hpi), triplicate (THEV-infected) and duplicate (mock-infected) samples were harvested for total RNA extraction.

### RNA Extraction and Sequencing     
Total RNA was extracted from infected cells using the Thermofisher RNAqueous™-4PCR Total RNA Isolation Kit (which includes a DNase I digestion step) per manufacturer's instructions. Agarose gel electrophoresis was performed to check RNA integrity (data not shown). The RNA quantity and purity was initially assessed using Nanodrop, and RNA was used only if the A260/A280 ratio was 2.0 ± 0.05 and the A260/A230 ratio was \>2 and \<2.2. Extracted total RNA samples were sent to LC Sciences, Houston TX for poly-A-tailed mRNA sequencing. RNA integrity was checked with Agilent Technologies 2100 Bioanalyzer High Sensitivity DNA Chip and samples with an RNA integrity number (RIN) \< 7 were excluded. Poly(A) RNA-seq library was prepared following Illumina's TruSeq-stranded-mRNA sample preparation protocol. Paired-end sequencing, generating 149 bp reads was performed on the Illumina NovaSeq 6000 sequencing system. The paired-end 149 bp sequences obtained during this study and all expression data have been submitted to the Gene Expression Omnibus database, under accession no GSE286211

### Quality Control and Mapping Process   
Sequencing reads were processed following a well-established protocol described by Pertea *et al* [@Pertea2016], using Snakemake - version 7.32.4 [@Snakemake2021], a popular workflow management system to drive the pipeline. Briefly, raw sequencing reads were trimmed with Cutadapt - version 1.10 [@Martin2011] and the quality of trimmed reads evaluated using the FastQC software, version 0.12.1 (Bioinformatics Group at the Babraham Institute, Cambridge, United Kingdom; www.bioinformatics.babraham.ac.uk), achieving an overall Mean Sequence Quality (PHRED Score) of 36. Trimmed reads were mapped the reference turkey (*Meleagris gallopavo*) genome file GCF_000146605.3_Turkey_5.1_genomic.fna.gz from NCBI (genome build: melGal5) (<https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/146/605/GCF_000146605.3_Turkey_5.1/>) with Hisat2 - version 2.2.1 [@Pertea2016] using the accompanying gene transfer format (GTF) annotation file (GCF_000146605.3_Turkey_5.1_genomic.gtf.gz) to build a genomic index. Samtools - version 1.21 was used to convert the output Sequence Alignment Map (SAM) file to the Binary Alignment Map (BAM) format. The StringTie (v2.2.1) software [@Pertea2016], set to expression estimation mode was used to generate normalized gene expression estimates from the BAM files for genes in the reference GTF file after which the prepDE.py3 script was used to extract read count information from the StringTie gene expression files, providing an expression-count matrix for downstream DEG analysis.

### DEG Analysis and Functional Enrichment Analysis     
DEG analysis between mock- and THEV-infected samples was performed using DESeq2 [@Deseq2], which employs a Negative Binomial distribution model for determining statistical significance when comparing read counts from multiple replicates. Genes with (FDR)-adjusted P-value $\leq$ 0.05 were considered as differentially expressed. The sequencing data (FASTQ files), expression-count matrix, and DEG analysis results from DESeq2 are deposited at NCBI Gene Expression Omnibus under accession number GSE286211. The functional profiling of DEGs (GO and KEGG analyses) were performed based on GO databases and KEGG databases using DAVID and the R package gprofiler2 [@Gprofiler2] with *M. gallopavo* as the reference organism. Results with P~adjusted~-value $\leq$ 0.05 were included as functionally enriched. All visualization plots were made using ggplot2, pheatmap, and ggvenn R packages [@ggplot2; @pheatmap; @ggvenn].

### Validation of DEGs by Reverse Transcriptase Quantitative PCR (RT-qPCR)   
The gene expression levels of representative DEGs (*APAF1, BMF, FADD, PDCD4, MADD, VCP, UFD1, EDEM1, EIF3D, EIF3M, RPL8, RPL10A*) were validated by quantification of relative mRNA levels with turkey *GAPDH* mRNA levels as the control gene. Briefly, the cells were infected and RNA extracted as described for the RNA sequencing samples with three biological replicates at 12- and 24-hpi each for both THEV-infected or mock-infected samples. First-strand cDNA synthesis of total RNA was performed with an oligo-dT primer to amplify poly-A-tailed mRNA using SuperScript™ IV First-Strand Synthesis System. The parent RNA was digested using RNase H after cDNA synthesis was complete to ensured that only cDNA remained as the template for the RT-qPCR quantification. The RT-qPCR was performed with the PowerUp™ SYBR™ Green master mix from Applied Biosystems with primers designed manually in the SnapGene software. The primers were checked for specificity using NCBI Nucleotide BLAST (https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn) before use. All primers used in this study are listed in **Supplementary Table S1**. Relative mRNA levels were calculated by 2^-$\Delta$$\Delta$CT^ method [@Livak2001].

### Statistical Analysis    
Statistical analyses of the RT-qPCR results were performed using R (Version 4.3.3) with Student's t-test and Mann-Whitney U test for the comparison between two groups. A p-value $\leq$ 0.05 was considered statistically significant.
\newpage

## 3. Results   
```{r mapping_stats}
source("scripts/r_code/reads_mapping_stats.R")

tot_raw_rds <- mapStats %>% pull(raw_reads) %>% sum(.)
tot_trm_rds <- mapStats %>% pull(trimmed_reads) %>% sum(.)
min_tr_rds <- mapStats %>% pull(trimmed_reads) %>% min(.)
max_tr_rds <- mapStats %>% pull(trimmed_reads) %>% max(.)

min_map_perc <- mapStats %>%
  mutate(map_perc = str_replace(mapped_reads, "\\d+\\.?\\d+\\s\\((\\d+\\.?\\d+)%\\)", "\\1")) %>%
  mutate(map_per = as.numeric(map_perc)) %>% pull(map_perc) %>%
  min(.) %>% as.numeric(.) %>% round(., 1)
max_map_perc <- mapStats %>%
  mutate(map_perc = str_replace(mapped_reads, "\\d+\\.?\\d+\\s\\((\\d+\\.?\\d+)%\\)", "\\1"),
         map_per = as.numeric(map_perc)) %>% pull(map_perc) %>%
  max(.) %>% as.numeric(.) %>% round(., 1)
```

**3.1.1 Sequencing Results**    
To identify the host transcriptomic profile during THEV infection, we infected MDTC-RP19 cells with THEV or no virus (mock) in triplicates or duplicates, respectively, and harvested at total RNA at 4-, 12-, 24-, and 72-hours post infection (hpi). mRNAs extracted from mock- or THEV-infected cells were sequenced on the Illumina platform, yielding a total of `r tot_raw_rds` million raw reads (149 bp in length) across all samples (see **Table 1** for sequencing statistics). After trimming low-quality reads, the remaining `r tot_trm_rds` million total paired-end trimmed reads (approximately, `r min_tr_rds`-`r max_tr_rds` million reads per sample) were mapped to the reference genome of *M. gallopavo* obtained from the National Center for Biotechnology Information (NCBI). The percentage of reads that mapped to the host genome across all samples ranged from `r min_map_perc` to `r max_map_perc`%. We observed that the fraction of reads that mapped to the host genome decreased while those mapping to the virus genome increased over the course of the infection as the viral infectious cycle progressed. Despite excellent quality scores at all time points (**Table 1**), DEGs identified at 4- and 72-hpi did not yield any statistically significant results in the downstream functional enrichment analyses (GO term and KEGG pathway analysis) and they were excluded from all subsequent analyses. In the remaining 12- and 24-hpi samples, a high consistency was observed between biological replicates (**Figure 2A** and **B**).

**3.1.2 DEGs of THEV-infected Versus Mock-infected Cells**   
```{r DEG_stats, message=F}
source("scripts/r_code/extract_deg_geneIDs.R")

# extracting counts for number of genes
tot_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>% nrow(.) %>% format(., big.mark = ",")
tot_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>% nrow(.) %>% format(., big.mark = ",")

up_12 <- all_deg_tables[["up_degIDs_12hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
down_12 <- all_deg_tables[["down_degIDs_12hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
up_24 <- all_deg_tables[["up_degIDs_24hrs"]] %>% nrow(.) %>% format(., big.mark = ",")
down_24 <- all_deg_tables[["down_degIDs_24hrs"]] %>% nrow(.) %>% format(., big.mark = ",")

# extracting log2FC ranges
min_fc_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  pull(log2fc) %>% min(.) %>% round(., 1)
min_fc_12_name <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  slice_min(log2fc) %>% pull(gene_name)
max_fc_12 <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  pull(log2fc) %>% max(.) %>% round(., 1)
max_fc_12_name <- diff_exp_genes %>% filter(timepoint == "t12") %>%
  slice_max(log2fc) %>% pull(gene_name)
min_fc_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  pull(log2fc) %>% min(.) %>% round(., 1) %>% format(., nsmall = 1)
min_fc_24_name <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  slice_min(log2fc) %>% pull(gene_name)
max_fc_24 <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  pull(log2fc) %>% max(.) %>% round(., 1)
max_fc_24_name <- diff_exp_genes %>% filter(timepoint == "t24") %>%
  slice_max(log2fc) %>% pull(gene_name)
```
We quantified gene expression levels with the StringTie software [@Pertea2016] in Fragments per kilobase of transcript per million (FPKM) units. The differential expression analysis of DEGs was performed with the DESeq2 R package [@Deseq2] which employs a negative binomial distribution model for determining statistical significance. Using a false discovery rate (FDR)-adjusted P-value cutoff $\leq$ 0.05 as the inclusion criteria, `r tot_12` and `r tot_24` genes from THEV-infected samples were identified as differentially expressed relative to their time-matched mock-infected samples at 12-hpi and 24-hpi, respectively. The results from the DEG analyses at 12- and 24-hpi have been deposited in NCBI Gene Expression Omnibus (<http://www.ncbi.nlm.nih.gov/geo>) under accession number GSE286211 with files named total_12hrsDEGs.csv.gz and total_24hrsDEGs.csv.gz, respectively. We compared THEV-infected samples relative to their time-matched mock-infected samples in identifying the significant DEGs and in the functional enrichment analyses. At 12-hpi (THEV-infected versus mock-infected), we found `r up_12` upregulated genes and `r down_12` downregulated genes, whereas `r up_24` genes were upregulated and `r down_24` genes downregulated at 24-hpi (THEV-infected versus mock-infected) (**Figure 2C**, and **Figure 3A-C**). The log~2~fold-change (FC) values at 12-hpi ranged between `r min_fc_12` and +`r max_fc_12` for `r min_fc_12_name` (Transmembrane Protein 156) and `r max_fc_12_name` (Lipase G), respectively. At 24-hpi, the log~2~FC values ranged between `r min_fc_24` and +`r max_fc_24` for `r min_fc_24_name` (C1q And TNF Related 12) and `r max_fc_24_name` (Potassium Voltage-Gated Channel Modifier Subfamily G Member 1), respectively.  
 
**3.1.3 Functional Enrichment Analyses (GO and KEGG pathway Analyses)**  
Gene ontology (GO) enrichment analysis was performed for the DEGs determined at the 12- and 24-hpi timepoints with the DAVID (Database for Annotation, Visualization and Integrated Discovery; version 2021) online resource [@Sherman2022] and the gprofiler2 R package -- version `r packageVersion("gprofiler2")` [@Gprofiler2], which outputs results according to the three branches of the GO directed acyclic graph -- cellular components (CP), biological processes (BP), and molecular functions (MF). We compared THEV-infected samples relative to their time-matched mock-infected samples for each timepoint. Results with P~adjusted~-value $\leq$ 0.05 were considered functionally enriched. The GO enrichment analyses results at 12-hpi and 24-hpi showed significant overlaps among all three GO categories. At both time points, cellular breakdown processes were upregulated while cellular maintenance processes and structures were downregulated in all three GO categories (**Table 2A-B** and **Table 3A-B**).

For upregulated DEGs at 12-hpi, we observed that the GO terms annotated under the BP category broadly cluster into: apoptosis and autophagy, cellular metabolism (catabolic processes), sterol biosynthesis, response to stimuli, and protein processing (**Figure 4A** and **Table 2A**). In the CC category, the GO terms relate primarily with cytoplasmic vacuolation, while in the MF category, they broadly fit under protein binding and kinase activity (**Table 2A**). For downregulated DEGs at 12 hpi, the GO terms in BP category generally fell under translation, protein biosynthesis and folding, ribosome biogenesis, nitrogen compound metabolism, nucleic acid synthesis, repair, metabolism, processing, and replication, and energy metabolism. Also, immunoglobulin production and isotype switching were downregulated (**Figure 4C** and **Table 2B**). In the CC category GO terms broadly grouped into ribosome, mitochondria, respirosome, nucleus, and spliceosome, while in the MF category, they generally belong to translation regulator activity, protein folding chaperone, catalytic activity (acting on a nucleic acids), and ATP hydrolysis activity (**Table 2B**).

At 24-hpi, we found that the GO terms in the BP category for upregulated DEGs were associated with apoptosis and autophagy, lipid and sterol biosynthesis, catabolic process, protein ubiquitination and proteolysis, cell signaling, and cell metabolism. Additionally, host defense response and genes that negatively regulate cytokine production were upregulated (**Figure 4B** and **Table 3A**). In the CC category, the GO terms were related to cytoplasmic vacuolation and the lysosome, similar to those identified at 12-hpi. In the MF category, the GO terms grouped into protein ubiquitination activity, kinase and acyltransferase activity, and macromolecule binding activity (**Table 3A**). GO terms for the downregulated DEGs were markedly similar to those at 12-hpi in all three GO categories. In the BP category, the GO terms broadly group into translation, peptide biosynthesis and folding, ribosome biogenesis, aerobic respiration and ATP synthesis, and cell cycle process and nucleic acid replication and processing (**Figure 4D** and **Table 3B**). The GO terms in the CC category group under ribosome, mitochondrion, nucleus and chromosomes, while the MF category, the GO terms grouped into structural components of ribosome and translation regulator activity, catalytic activity acting on a nucleic acid and nucleic acid binding, aminoacyl-tRNA ligase activity, and NAD binding (**Table 3B**).

KEGG pathway analysis on the DEGs was also performed using both the gprofiler2 R package [@Gprofiler2] and the DAVID online resource. Both resources gave similar results, but the results from DAVID (**Table 4**) included more information than the gprofiler2 results (**Table S2**). The results from the KEGG pathway analysis were consistent with the GO results, revealing that generally, cell maintenance and upkeep pathways were downregulated while cell death and breakdown pathways were upregulated. We observed that cell maintenance pathways such as DNA replication and repair, ribosome biogenesis, spliceosome, and oxidative phosphorylation were downregulated at both 12- and 24-hpi. Pathways such as: autophagy, response to virus (Influenza A), and steroid biosynthesis were upregulated at 12-hpi, which is similar to 24-hpi, where pathways such as: autophagy, ubiquitin-mediated proteolysis, lysosome, protein processing in endoplasmic reticulum, and steroid biosynthesis were upregulated.

It is well-established that THEV induces cell death (apoptosis and necrosis) in infected B-cells, which is linked to THEV-induced IMS [@Rautenschlein2000b; @Rautenschlein2000; @Saunders1993]. Hence, we were particularly interested in cellular processes and pathways associated with cell death and pathways that may affect the survival of the host B-cells, thereby accounting for THEV-induced IMS. We highlight the upregulated cell death (apoptosis and autophagy), ubiquitin-dependent endoplasmic reticulum [ER]-mediated protein degradation, and suppressed cell maintenance pathways as well as cytokine deregulation identified by our GO and KEGG analyses as the likely key aspects of THEV-host cell interaction relevant to THEV-induced IMS. 

**3.1.4 Cell Death and Breakdown Pathways Upregulated by THEV**   
Many virus families, including adenoviruses, herpesviruses, poxviruses, baculoviruses, parvoviruses, retroviruses, rhabdoviruses, paramyxoviruses, orthomyxoviruses, togaviruses, and picornaviruses are known to trigger apoptosis in infected host cells either through direct viral protein action or the host antiviral response [@Barber2001; @Hardwick1997; @Verburg2022]. Our data show that apoptotic and autophagic pathways are upregulated during THEV infection, supporting previous findings of apoptosis and necrosis of THEV-infected cells [@Rautenschlein2000b; @Rautenschlein2000; @Saunders1993]. For example, several proapoptotic members of the BCL2 (B-cell lymphoma 2) protein family such as BCL2 antagonist/killer 1 (*BAK1*), BCL2 interacting protein 3 like (*BNIP3L*), BCL2 interacting protein 3 (*BNIP3*), and Bcl2 modifying factor (*BMF*) were upregulated. Additionally, Fas cell surface death receptor (*FAS*), Fas associated via death domain (*FADD*), MAP kinase-activating death domain (*MADD*), programmed cell death 4 (*PDCD4*), RB1 inducible coiled-coil 1 (*RB1CC1*), activating transcription factor 4 (*ATF4*), receptor interacting serine/threonine kinase 1 (*RIPK1*), tumor necrosis factor receptor superfamily member 1B (*TNFRSF1B*), pro-apoptotic WT1 regulator (*PAWR*), and apoptotic peptidase activating factor 1 (*APAF1*), which are potent proapoptotic factors were upregulated at both timepoints. Interestingly, both the intrinsic (*BAK1, BNIP3L, BNIP3, BMF, RB1CC1, ATF4, PDCD4*, and *APAF1*) and extrinsic (*FAS, FADD, TNFRSF1B, MADD, and RIPK1*) apoptotic pathways were represented. Conversely, several anti-apoptotic proteins such as BCL2 apoptosis regulator (*BCL2*), BCL2 interacting protein 2 (*BNIP2*; interacts directly with adenovirus E1B-19K protein), BCL2 related protein A1 (*BCL2A1*), and apoptosis inhibitor 5 (*API5*) were also upregulated. Thus, apoptosis and its regulation pathways are clearly upregulated; this highlights the host-virus tug-of-war also typical in Mastadenovirus infections. Moreover, several genes associated with autophagy such as: TNF receptor associated factor 6 (*TRAF6*), autophagy related 9A (*ATG9A*), unc-51 like autophagy activating kinase 2 (*ULK2*), and autophagy related 4B cysteine peptidase (*ATG4B*) were upregulated.

**3.1.5 Downregulation of Cell Maintenance Pathways**   
Forcibly transitioning the host cell cycle to the S phase during the early phase of infection is a prerequisite for a productive adenovirus infection [@Zhao2012]. Interaction of the viral E1A early proteins with the host pRb (retinoblastoma) protein releases the host transcription factor E2F, which activates genes required for S phase cell cycle induction. Viral E1A also binds the host transcriptional co-activator p300/CBP [@Guimet2016; @Zhao2012]. Our GO and KEGG pathway results showed that at 12-hpi, several key genes involved with cell cycle transition were upregulated. Notably, E1A binding protein p300 (*EP300*), cyclin genes (*CCND3, CCNG1, CCNG2, CDK6*), anaphase promoting complex subunit 1 (*ANAPC1*), and cell division cycle 27 (*CDC27*) were upregulated. However, unlike the observation in Mastadenoviruses, the cell cycle regulation at 12- and 24-hpi seem complicated as some key cell cycle-related genes and general cell maintenance processes were concurrently downregulated.

We found that several essential cell maintenance processes whose suppression can trigger apoptosis were downregulated. Severe DNA damage is a known mechanism of apoptosis induction, called DNA damage-dependent apoptosis [@Roos2006]. Repression of host RNA and protein synthesis is also strongly associated with apoptosis [@Martin1993]. Several processes related to DNA and RNA synthesis, maintenance, and repair such as nucleotide biosynthesis and metabolism, double strand break repair, DNA excision repair, RNA biosynthesis, RNA processing, DNA replication, mitotic cell cycle process, protein-RNA complex organization, and DNA damage response were downregulated at both timepoints. Notable genes identified include DNA ligase 1 (*LIG1*), X-ray repair cross complementing 1 (*XRCC1*), cyclin dependent kinase 1 and 2 (*CDK1, CDK2*), checkpoint kinase 1 (*CHEK1*), 8-oxoguanine DNA glycosylase (*OGG1*), BLM RecQ-like-helicase (*BLM*), BRCA1 DNA repair associated (*BRCA1*), and several RAD family proteins (*RAD21, RAD51, RAD51B, RAD51C, RAD54B*). 

Protein synthesis-related  processes, including ribosome biogenesis, rRNA processing, ribosome assembly, protein folding, translational initiation, protein maturation, ribosome and ribonucleoprotein complex formation, translation pre-initiation complex formation, and cytoplasmic translation were significantly downregulated at both 12- and 24-hpi. Notable genes identified include eukaryotic translation initiation factors (*EIF1, EIF1AX, EIF3E and EIF3F, EIF3H, EIF3I, EIF3L and EIF3M*), biogenesis of ribosomes BRX1 (*BRIX1*), MCTS1 re-initiation and release factor (*MCTS1*), and ribosomal protein subunits (*RPL8, RPL10a, RPL11, RP12, RP13, RP14, RP15, RP18a, RP19*).

**3.1.6 Endoplasmic Reticulum (ER) Stress Response during THEV infection**   
Our KEGG pathway analysis (**Table 4**) showed that protein processing in the ER, and ubiquitin-mediated proteolysis are significantly upregulated (**Figure 5**). The GO results (**Table 3A**) showed that specifically, ER stress and the ER‐associated protein degradation (ERAD) pathway, a branch of the unfolded protein response (UPR) were upregulated during THEV infection, especially at 24-hpi. The ER is the major site for protein synthesis, folding and quality control, and sorting [@Christianson2022]. Upon ER stress or continued accumulation of unfolded proteins in the ER lumen, the UPR pathways are activated to restore ER homeostasis. The ERAD pathway, a ubiquitin‐proteasome‐dependent pathway, is a protein quality control system activated for degradation of misfolded and unassembled proteins [@Christianson2022]. In our results, the THEV-infected samples showed significant increase in ERAD pathway effector proteins, such as valosin containing protein (*VCP*), ubiquitin recognition factor in ER associated degradation 1 (*UFD1*), ER degradation enhancing alpha-mannosidase like proteins 1 and 3 (*EDEM1, EDEM3*), cullin 1 (*CUL1*), and ubiquilin 1 (*UBQLN1*). Other genes related to other UPR pathways such as *HSPA5* and *ATF4* were also upregulated. Our KEGG pathway (**Table S2**) and GO (**Figure 4B**) results indicated a significant upregulation of ubiquitin-mediated proteolysis with other ubiquitination pathway proteins such as ubiquitin conjugating enzymes (*UBE2J2, UBE2E3, UBE2Z*), ubiquitin protein ligases (*UBE3A, UBE3B*), NPL4 homolog ubiquitin recognition factor (*NPLOC4*), and ubiquitin-like modifier activating enzyme 6 (*UBA6*) showing significant upregulation. Additionally, the heat shock family of chaperone proteins such as the DnaJ heat shock protein family (*HSP40*) members (*DNAJB11, DNAJB12, DNAJB2, DNAJC10*),  heat shock protein family A (*HSP70*) members (*HSPA4L, HSPA5, HSPA8*), and heat shock protein 90 alpha family class A member 1 (*HSP90AA1*) were upregulated at 24-hpi. Moreover, the KEGG pathway analysis (**Table 4**) shows a significant upregulation in lysosome formation, lumen acidification, and lysosomal degradation, likely an indication of ER-to-lysosome-associated degradation. Taken together, these results suggest that THEV infection triggers significant ER-associated protein degradation, which may contribute to cell death and IMS.

**3.1.7 Differential Expression of Cytokine and Cytokine Receptor-encoding Genes**   
Our KEGG pathway results showed that a pathway similar to immune response to influenza A infection was upregulated at 12-hpi. Our GO analysis also identified terms such as regulation of lymphocyte activation and regulation of cytokine production as upregulated at both 12- and 24-hpi. Genes involved include *IL18, IL2RB, IL4R, IL5RA*, TNF receptor associated factors (*TRAF2, TRAF3, TRAF6, TRAF7, TRAFD1*), TNF receptor superfamily members (*TNFRSF1B, TNFRSF8, TNFSF4*), interferon-induced with helicase C domain 1 (*IFIH1*), interferon-induced double-stranded RNA-activated protein kinase (*PKR*), and	*CD80*. In contrast, cytokine inhibitors such as suppressor of cytokine signaling (*SOCS3* and *SOCS5*) were also upregulated at both 12 and 24-hpi and immunoglobulin production and isotype switching GO terms were downregulated at 12-hpi. This inconsistency is likely an indicator of the struggle between the virus and its host. While several cytokines were regulated by THEV as in the proposed model of THEV immunopathogenesis (**Figure 1**), the cytokines in the model (IFN-\(\alpha\), IFN-\(\beta\), IFN-\(\gamma\) TNF, and IL-6) were not significantly differentially expressed in our data. However, some of the differentially expressed cytokines and cytokine receptors (*TNFRSF8, TRAF7*) identified in this study are positive regulators of apoptosis; therefore, they may play a role in THEV-induced IMS.

**3.1.8 Validation of DEGs by Reverse Transcriptase Quantitative PCR (RT-qPCR)**    
To validate the RNA-seq results, 12 DEGs (8 upregulated and 4 downregulated) were selected for RT-qPCR. The DEGs were representative of apoptosis (*APAF1, BMF, FADD, MADD, and PDCD4*), ERAD and ubiquitination (*VCP, UFD1, EDEM1*), and ribosome biosynthetic (*EIF3D, EIF3M, RPL8, RPL10A*) pathways. As shown in **Figure 6**, the RT-qPCR results corroborate the RNA-seq results, further reinforcing the validity of the RNA-seq transcriptomic profile results. Although there was no inconsistency between the RNA-seq and RT-qPCR results in terms of gene regulations, the fold changes in the RT-qPCR results were consistently higher than observed in the RNA-seq results. Our RT-qPCR primers showed excellent target specificity; only one amplicon of the expected size was amplified as shown by the melt curves (data not shown) and gel electrophoresis (**Figure S1**). According to our Student's T-test and Mann-Whitney U test, the difference in gene expression levels in all the selected genes were statistically significant.
\newpage

### 3.2 Tables and Figures

![**Figure 1.** Model of THEV-induced immunosuppression in turkeys. THEV infection of target cells is indicated with black dotted arrows. Black unbroken arrows indicate cell activation. Red arrows indicated signals leading to cell death (apoptosis/necrosis). Blue arrows indicate all cytokines released by the cell. Blue arrows with square heads indicated an event leading to IMS. Adapted from Rautenschlein *et al.* [@Rautenschlein2000].](results/imss_model.png)

![**Figure 2.** (A) Principal component analysis (PCA) of turkey B-cells during THEV infection. At 12-hpi (**A1**), the results indicate that the first (PC1) and second (PC2) principal components account for 96% and 3% of the variation in the samples, respectively. Whereas PC1 and PC2 account for 96% and 2% of the variation, respectively at 24-hpi (**A2**). (**B**) Poisson distance matrices illustrating the RNA-seq library integrity within treatment (infected versus mock) groups. The color scale represents the distances between biological replicates for both 12-hpi samples (**B1**) and 24-hpi samples (**B2**). Dark colors represent high correlation (similarity) between the samples involved. (**C**) Volcano plots of DEGs between THEV-infected versus mock-infected cells at 12- and 24-hpi. Red, blue, and grey dots represent upregulated, downregulated, and non-significant genes, respectively for both 12-hpi samples (**C1**) and 24-hpi samples (**C2**).](results/r/figures/sample_corr_figure.png)

![**Figure 3.** DEGs of THEV-infected versus mock-infected samples at different time points. (**A**) Bar plot of number DEGs identified. Red represents upregulated genes and blue represents downregulated genes.  (**B**) Heatmaps of scaled expression data (Z-scores) of DEGs. DEGs identified at 12-hpi are shown in (**B1**) and DEGs at 24-hpi in (**B2**). (**C**) Venn diagrams showing the number of DEGs identified at different time points. For the upregulated genes (**C1**), the blue circle represents genes at 4-hpi, the yellow circle, 12-hpi, and the green circle, 24-hpi. For the downregulated genes (**C2**), the red circle represents genes at 72-hpi, while all the other time points retain the colors from (**C1**).](results/r/figures/deg_patch_fig.png)

![**Figure 4.** Dotplot of Enriched Gene Ontology Biological Processes (BP). Significant BP GO terms identified for upregulated DEGs at 12-hpi and 24-hpi are shown in (**A**) and (**B**), respectively. Significant BP GO terms for downregulated DEGs at 12-hpi and 24-hpi are shown in (**C**) and (**D**), respectively. The y-axis indicates GO terms and the x-axis represents the rich factor, which indicates the ratio of the number of DEGs annotated to the term to the total number of genes annotated to the term. The diameter indicates the number of genes overlapping the gene ontology term and the color indicates the enrichment P-value.](results/r/figures/patch_GO_enrich.png)

![**Figure 5.** Upregulation of ER Unfolded Protein Response (UPR). KEGG Pathway analysis shows  multiple key genes involved in the ER UPR were upregulated. All genes from our DEG list are annotated with the red star. Notably, *ATF4, PERK, VCP (p97), TRAF2, UFD1* and several BCL2 and heat shock proteins are upregulated. We see that the PERK branch of the UPR pathway linked to apoptosis is upregulated. Another pathway linked to apoptosis via *BAX* is shown as well as the ERAD protein degradation pathway. Note that due to limited annotation of the host genome, a significant proportion of the DEGs were not recognized by the database; hence not shown here. _Figure generated from KEGG pathway analysis in DAVID_.](results/upr_pathway.png)

![**Figure 6.** Validation of representative DEGs involved in Apoptosis, Protein synthesis, and ER-stress responses by RT-qPCR. MDTC-RP19 cells infected with THEV- or mock-infected were subjected to RT-qPCR analysis for the relative expression of the indicated DEGs at 24-hpi. *GAPDH* was used as the internal control. Data are expressed as the mean $\pm$ SD. All genes (THEV-infected) are statistically differentially expressed relative to their time-matched mock-infected counterparts based on Student's t-test and Mann-Whitney U test.](results/r/figures/qpcr_validation.png)

\newpage
\FloatBarrier
```{r, mapping_stats_table}
set_flextable_defaults(font.size = 8, table.layout = "fixed", float = "none")
mapStats %>%
  flextable(., cheight = 0.1) %>%
  theme_box() %>%
  set_header_labels(sample = "Sample", raw_reads = "Raw Reads", trimmed_reads = "Trimmed Reads",
                    mapped_reads = "Mapped Reads", unq_map_reads = "Uniquely Mapped \nReads",
                    multi_map_reads = "Non-uniquely \nMapped Reads", q20 = "Q20%",
                    q30 = "Q30%", gc_content = "GC\nContent (%)") %>%
  flextable::font(part = "all", fontname = "Times New Roman") %>%
  flextable::bold(part = "header") %>%
  flextable::set_table_properties(layout = "fixed", align = "center",
                                  opts_pdf = list(caption_repeat = F),
                                  opts_word = list(split = F, keep_with_next = T)) %>%
  flextable::align(align = "center", part = "all") %>%
  footnote(., j = c(2:6),
           value = as_paragraph(c("All values for number of reads are in millions")),
           ref_symbols = c("M"), part = "header", inline = T) %>%
  footnote(., j = 1, i = c(1:11),
           value = as_paragraph(c("These are infected samples indicated by the letter 'I' and 'S' in sample names")),
           ref_symbols = c("Inf"), part = "body", inline = T) %>%
  footnote(., j = 1, i = c(12:19),
           value = as_paragraph(c("These are mock-infected samples indicated by the letters 'U' and 'N' in sample names")),
           ref_symbols = c("Mk"), part = "body", inline = T) %>%
  set_caption("Table 1: Summary of sequencing, quality control, and mapping processes")

```
\FloatBarrier
\newpage
```{r GOTerm_Enrich_UP_12hpi}
source("scripts/r_code/generate_GO_tables.R")
# Table 2A
make_GO_table <- function(goTerms_res){
  # Calculate row number to merge GOTerm category label
  bp_row_num <-  goTerms_res %>%
    filter(source %in% c("GO:BP", "Biological Process")) %>% nrow(.) + 1
  cc_row_num <- goTerms_res %>%
    filter(source %in% c("GO:CC", "Cellular Component")) %>% nrow(.) + bp_row_num
    
  table <- goTerms_res %>%
    arrange(source, term) %>%
    mutate(source = case_when(is.na(fold_enrich) & source == "GO:BP" ~ "Biological Process",
                              is.na(fold_enrich) & source == "GO:CC" ~ "Cellular Component",
                              is.na(fold_enrich) & source == "GO:MF" ~ "Molecular Function",
                              TRUE ~ source)) %>%
    flextable() %>%
    theme_box() %>%
    set_header_labels(source = "GO Category", term = "GO:Term",
                      qval = "P-value\n(Adjusted)",
                      num_degs = "Number of DEGs",
                      fold_enrich = "Fold Enrichment") %>%
    flextable::font(part = "all", fontname = "Times New Roman") %>%
    flextable::bold(part = "header") %>%
    flextable::set_table_properties(layout = "fixed", align = "center",
                                    opts_pdf = list(caption_repeat = F),
                                    opts_word = list(split = F, keep_with_next = T)) %>%
    flextable::align(align = "center", part = "all") %>%
    merge_at(., i = 1, j = 1:5, part = "body") %>%
    merge_at(., i = bp_row_num, j = 1:5, part = "body") %>%
    merge_at(., i = cc_row_num, j = 1:5, part = "body") %>%
    flextable::bold(part = "body", i = c(1, bp_row_num, cc_row_num)) %>%
    flextable::fontsize(part = "body", size = 8) %>%
    flextable::fontsize(part = "head", size = 8) %>%
    flextable::fontsize(part = "body", i = c(1, bp_row_num, cc_row_num), size = 8) %>%
    flextable::width(., j = 2, width = 3.5)
  
  return(table)
}

go_res_table12up <- tab_res %>% filter(timepoint == "t12") %>% select(go_tab_up) %>% unnest(.)

make_GO_table(go_res_table12up) %>%
    set_caption("Table 2A: Gene ontology analysis of Significantly Upregulated DEGs identified at 12-hpi")

```
\FloatBarrier
\newpage

```{r GOTerm_Enrich_DOWN_12hpi}
# Table 2B
go_res_table12dw <- tab_res %>% filter(timepoint == "t12") %>% select(go_tab_down) %>% unnest(.)

# write_tsv(go_res_table12dw, "results/r/tables/submission/table2b.tsv",
#           col_names = T)

make_GO_table(go_res_table12dw) %>%
    set_caption("Table 2B: Gene ontology analysis of Significantly Downregulated DEGs identified at 12-hpi")

```
\FloatBarrier
\newpage

```{r GOTerm_Enrich_UP_24hpi}
# Table 3A
go_res_table24up <- tab_res %>% filter(timepoint == "t24") %>% select(go_tab_up) %>% unnest(.)

make_GO_table(go_res_table24up) %>%
    set_caption("Table 3A: Gene ontology analysis of Significantly Upregulated DEGs identified at 24-hpi")

```
\FloatBarrier
\newpage

```{r GOTerm_Enrich_DOWN_24hpi}
# Table 3B
go_res_table24dw <- tab_res %>% filter(timepoint == "t24") %>% select(go_tab_down) %>% unnest(.)

make_GO_table(go_res_table24dw) %>%
    set_caption("Table 3B: Gene ontology analysis of Significantly Downregulated DEGs identified at 24-hpi")
```
\FloatBarrier
\newpage

```{r KEGG_result_DAVID}
# Load script with KEGG result tables
source("scripts/r_code/process_DAVID_kegg.R")

david_kegg <- kegg_result %>%
  mutate(across(c(num_degs:qval), as.numeric),
         qval = format(qval, digits = 3)) %>%
  select(-c(rich_factor, pval)) %>%
  mutate(regulation = str_replace(timepoint, "^((down|up))\\d{2}hrs$", "\\1"),
         timepoint = str_replace(timepoint, "^(down|up)(\\d{2})hrs$", "\\2-hpi")) %>%
  select(timepoint, regulation, everything())
  

david_kegg %>% flextable(.) %>%
  theme_box() %>% 
  width(., j = "term", width = 2.5) %>%
  set_header_labels(timepoint = "Time Point", regulation = "Regulation",
                    term = "KEGG Term", num_degs = "DEG Count",
                    fold_enrich = "Fold Enrichment", qval = "P-value\n(Adjusted)") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::fontsize(part = "all", size = 8) %>%
  set_caption("Table 4: Significantly Enriched KEGG Pathways from DEGs identified at 12- and 24-hpi (Results from the DAVID online resource)")

```
\newpage

## 4. Discussion      
THEV has a worldwide distribution, wreaking economic havoc on affected poultry farms, particularly due to its immunosuppressive trait allowing secondary opportunistic infections to devastate turkey populations [@Tykaowski2019; @Beach2006]. HE in turkeys causes more economic losses than any disease caused in other birds like chicken or pheasants [@Beach2006]. While the current vaccine strain (VAS) has proven effective at preventing clinical HE in turkey poults, the retention of its immunosuppressive properties leaves some of the issues associated with economic losses unresolved. Elucidating the virus-host interactions leading to IMS is most pressing for not only the understanding of the viral infection and pathogenesis but also future antiviral therapy targets. Since both virulent and avirulent THEV cause IMS but the avirulent are used as vaccine, we believe that studying VAS would be more expedient for understanding THEV vaccine-induced IMS. 

Only one cell line (MDTC-RP19 also known as RP19) has been found to be capable of supporting THEV replication [@vandenHurk1990]. Thus, in this work, we establish the first transcriptomic profile of THEV infection in RP19 cells using paired-end RNA-seq. We attempted a multi-time point experimental design but this being the first transcriptomic study of THEV infection, we faced some difficulties, including selecting our sampling time points based on the only study of THEV gene expression kinetics [@Aboezz2019], leading to only 12- and 24-hpi providing useful data. In total `r tot_12` and `r tot_24` DEGs were identified at 12-hpi and 24-hpi, respectively. At 12-hpi, `r up_12` genes were upregulated and `r down_12` genes downregulated, whereas `r up_24` genes were upregulated and `r down_24` genes downregulated at 24-hpi. Being a non-model organism, a significant proportion of the host (*M. gallopavo*) genes are not annotated and not recognized by the databases used for functional enrichment analysis. Thus, the obtained results are likely sub-optimal in amount of detail relative to results from well annotated and curated genomes of model organisms. The DEGs were related to multiple biological processes all potentially playing a role in THEV infection but the most relevant to our study are apoptosis (upregulated), ER stress-induced unfolded protein response (upregulated), cell maintenance processes (downregulated), and cytokine functions (deregulated). Furthermore, the RT-qPCR results validated the RNA-seq results. Collectively, this study may shed light on some significant aspects of THEV-host interactions, which may benefit further mechanistic delineation of the viral infection and induction of IMS and inform future development of anti-THEV strategies. The biological processes most relevant to THEV-induced IMS highlighted by this study are further discussed below.

Apoptosis is a key defense mechanism activated by cells in response to irreversible injury and virus infection to abrogate virus propagation. It is a formidable cellular defense network, non-specific to any virus family and therefore an important problem for any infecting virus to overcome [@Barber2001; @Hardwick1997; @Verburg2022]. The adenovirus E1A proteins are strong inducers of apoptosis. They bind host pRb and p300/CBP protein, inducing p53-mediated apoptosis, and can also sensitize infected cells to TNF$\alpha$ and TRAIL-induced apoptosis [@Zhao2012; @Guimet2016]. However, adenoviruses have developed multiple distinct anti-apoptotic mechanisms to counter almost all cellular pro-apoptotic programs. For example, E1A blocks its own induction of p53-dependent apoptosis and E1B proteins (E1B-19K and E1B-55K) counteract several types of apoptosis including TNF-induced apoptosis [@Zhao2012; @Guimet2016]. Despite the rich arsenal of countermeasures, transcriptomic studies of human adenovirus infections suggest a complex set of virus-host interactions where both pro- and anti-apoptotic genes are turned on contemporaneously. For example, in human adenovirus 2 infection, both pro- and anti-apoptotic BCL2 family genes were stimulated [@Zhao2012]. Siadenoviruses including THEV are the smallest adenoviruses and therefore encode the fewest genes [@Beach2009a; @Quaye2024]. THEV encodes a mere 34 ORFs with no anti-apoptotic genes characterized [@Quaye2024]. In agreement with these findings, in our results a strong signal indicative of apoptotic induction was observed. However, like Mastadenovirus infections, a complex relationship between pro and anti-apoptotic genes were observed. Pro-apoptotic genes such as *APAF1, BNIP3L, BMF, BAK1, RIPK1, FAS, FADD* and *ATF* were  upregulated in concert with the anti-apoptotic genes *BCL2, BNIP2, BCL2A1* and *API5*. We speculate that this complex regulation is predictive of THEV possessing some anti-apoptotic genes but not sufficiently potent to thwart the cellular apoptotic response. It is also possible that using a naturally attenuated strain may account for the apparent balance in pro- and anti-apoptotic signals. Interestingly, pro-apoptotic genes in both intrinsic and extrinsic pathways were upregulated, possibly due to a concurrent stimulation of multiple apoptotic pathways or a positive feedback mechanism of one system activating the other. The specific mechanism of apoptosis induction remains elusive. Further studies designed to elucidate these fine details are warranted and would benefit future THEV therapeutics.

The ER serves many specialized functions including biosynthesis and assembly of membrane and secretory proteins, calcium storage, and biosynthesis of lipids and sterols. It is also the site of protein folding and post-translational modifications and maintains stringent quality control systems, ensuring exported proteins are correctly folded and degradation of unfolded or misfolded proteins [@Fribley2009; @Mo2023; @Christianson2022]. Disruption of ER homeostasis or ER stress leads to accumulation of incorrect proteins in the ER lumen, triggering the UPR. The UPR restores ER normality by transiently attenuating general protein synthesis, increasing the lumenal folding capacity, and the degradation of misfolded proteins through the ERAD pathway or autophagy [@Fribley2009; @Mo2023; @Christianson2022; @Read2021]. However, if incorrect lumenal protein overload persists, the prolonged UPR will induce apoptosis known as ER stress-associated programmed cell death [@Fribley2009; @Read2021]. Many DNA and RNA viruses are reported to induce ER stress and UPR pathways during infection [@Mo2023]. In our results, *ATF4* and PKR-like ER protein kinase (*PERK*), key proteins in the *PERK* branch of the UPR pathway were upregulated. A myriad of ERAD pathway proteins (e.g., *VCP, UFD1, EDEM1, EDEM3, CUL1, UBQLN1*), ubiquitination system proteins (e.g., *UBE2J2, UBE2E3, UBE2Z, UBE3A, UBE3B*), and  heat shock family of chaperone proteins (e.g., *HSPA5, HSP4L, HSPA8, HSP90AA1*) all showed increased expression according to our RNA-seq data with some validated with RT-qPCR. These data strongly suggest that THEV infection triggers the ER UPR pathways leading to a massive decrease of protein synthesis and deregulation of sterol biosynthesis, and ubiquitin-mediated proteolysis, all supported by our results. As noted above, a prolonged UPR activation leads to ER stress-associated programmed cell death via genes such *ATF4* [@Fribley2009; @Read2021]. Thus, we suggest that the ER stress response likely plays a crucial role in THEV-induced IMS. Nonetheless, the mechanisms underlying the regulation of the UPR pathways by THEV remain to be clearly unraveled. Also, whether and how the ER stress response affects THEV infection and pathogenicity should be studied. Unsurprisingly, protein degradation was more evident at 24-hpi than at 12-hpi, reflecting the suggested two phases of UPR -- Phase 1 allows the unfolded proteins time to refold without degradation and Phase 2 degrades any proteins which have failed to fold [@Read2021]. 

In the proposed model of THEV immunopathogenesis by Rautenschlein *et al* (**Figure 1**), while THEV directly induced cell death in infected cells, cytokines are responsible for extending cell death to bystander splenocytes [@Rautenschlein2000]. However, the primary cytokines (IFN-\(\alpha\), IFN-\(\beta\), IFN-\(\gamma\) TNF, IL-6, and NO) highlighted in the model were not significantly differentially expressed in our data. This may be explained by the fact that the model was proposed based on data from splenocytes of THEV-infected turkeys, which have the full complement of immune cells (T-cells, B-cells, macrophages) shown in the model and not solely from B-cell culture data as in this study. From the model, T-cells and macrophages are the principal producers of the effector cytokines; thus, there is agreement with our data that B-cells alone would poorly recapitulate the cytokine communication network. This may also explain the very few immune-associated biological processes in our data as the B-cells may require cytokines from other immune cells such as macrophages and T-cells for optimal activation. Further transcriptomic studies with splenocytes would offer a wealth of insights regarding these ideas. It also likely that cytokines may only play a dominant role in some aspects of THEV-infection such as the clinical hemorrhage of the intestines but not the associated IMS since a study using the TNF-blocking drug (thalidomide) only prevented intestinal disease, not IMS [@Rautenschlein2000]. While some of the upregulated cytokines and receptors in our results are positive apoptosis regulators (*TNFRSF8, TRAF7*), most of the cytokines are either anti-apoptotic (*TNFRSF1B, TRAF2*), boost host antiviral defense (*IL18, TNFSF4, PKR, TRAFD1, IFIH1*), or suppress cytokine signaling (*SOCS3, SOCS5*). Therefore, we speculate that a non-cytokine-mediated apoptotic process such as ER stress-associated programmed cell death is more likely to mediate direct killing of infected cells. However, whether bystander cell death occurs and if it is cytokine-mediated as suggested by Rautenschlein *et al* are important questions that can be addressed with future transcriptomic studies in splenocytes.

By convention, the Mastadenovirus replication cycle is divided into two phases, an early and a late phase, separated by the onset of viral DNA replication [@Guimet2016; @Zhao2012]. Based on DNA microarray analysis, adenovirus type 2 (Ad2) infection has been divided into four stages. The first period is from 0 to 12-hpi during which changes in cellular gene expression are likely to be triggered by the viral entry process. Most of the deregulated genes have functions linked to inhibition of cell growth. Therefore, growth suppression is most likely the first response of the host cell to the incoming virus [@Zhao2012]. The second period covers the time from 12 to 24-hpi and follows activation of the immediate early E1A gene, which forcibly transition cell cycle to S phase [@Zhao2012]. While the temporal changes of host gene expression for a THEV infection have not been characterized in prior studies, our data suggest that during the first 24-hpi, cell growth was suppressed. Cell maintenance processes involving nucleic acid and proteins were downregulated according to our data. Protein synthesis-related processes including ribosome biogenesis, rRNA processing, ribosome assembly, protein folding, translational initiation, protein maturation, and others were heavily affected. Additionally, DNA and RNA synthesis, maintenance, and repair such as nucleotide biosynthesis and metabolism, double strand break repair, and DNA excision repair were also repressed. As severe DNA damage leads to DNA damage-dependent apoptosis [@Roos2006] and repression of RNA and protein synthesis is also strongly associated with apoptosis [@Martin1993], these inhibitions may also play a role in THEV-induced IMS. Moreover, we speculate that the ER UPR may contribute partly to the severe repression of protein synthesis as discussed above. An in-depth study of temporal changes of host gene expression during THEV infection would be invaluable in establishing if THEV follows the same pattern as Ad2.

## 5. Conclusions   
THEV-induced IMS is a pressing concern for turkey farmers worldwide, causing substantial economic losses annually. In this study, we establish the cellular transcriptomic profile of THEV infection in turkey RP19 B-cells using paired-end RNA-seq, identifying `r up_12` upregulated genes and `r down_12` downregulated genes at 12-hpi and `r up_24` upregulated genes and `r down_24` downregulated genes at 24-hpi. Our data suggest that several biological processes and pathways including apoptosis, immune response, ER response to stress, ubiquitin-dependent proteolysis, and repression of essential cellular maintenance are significant aspects of host cell response to THEV infection. All these processes are established apoptosis inducing mechanisms; therefore, we believe that either one or synergistic interplay between multiple ones may mediate cell death of infected B-cells, leading to IMS. These findings provide the first insights into THEV-host interactions and may help advance the understanding of non-human adenoviral infection and pathogenesis, which may eventually inform the development of medical countermeasures for disease prevention and treatment.

\newpage
**Supplementary Materials**

![**Figure S1: Gel Electrophoresis of RT-qPCR validation reactions.** We run a gel electrophoresis of the RT-qPCR reactions in 2% agarose gel to confirm primer specificity. The 13 primer pairs all show excellent target specificity, each amplifying one amplicon of the expected size (see Table S1 for expected amplicon sizes of each primer pair). This was also confirmed in the RT-qPCR melt curves (not shown). Thermo Scientific™ generuler 1 kb plus DNA ladder was used. Mock-infected samples are shown in the top row prefixed with "mock" and infected samples are shown in the bottom row prefixed with "inf".](qpcr_validation/qpcr_gel.png){width=90%}

\FloatBarrier
\newpage
```{r table S1}
# candidate house-keeping genes: TATA-box binding protein (TBP), GAPDH, actin beta (ACTB)
# TBP is not a DEG
tibble(gene_id = c("100303685", "100303677", "100543138", "100549497",
                   "100550591", "100540536", "100547583", "100551463",
                   "100548376", "104913522", "100543065", "100545922",
                   "100545633", "100544011", "100544053"),
       target = c("GAPDH", "ACTB", "TBP", "APAF1", "BMF", "FADD", "PDCD4", "MADD",
                  "VCP", "UFD1", "EDEM1", "EIF3D", "EIF3M", "RPL8", "RPL10A"),
       forwardP = c("CACTATCTTCCAGGAGCGTGACC", "CACGGCATTGTCACCAACTGG",
                    "CTCAGGGTGCAATGACTCCTGG", "GCTGCGCAAATACCCGAGGTC",
                    "CGGAGACTCTTCTATGGGAATGCTGG", "GGAGCTCTGCAACTTCCTCATGG",
                    "GCACAGTAGAAGTGGAGAATCTGAGTG", "GAGCTGACGAGGTTGAACTTGCTG",
                    "CAAGGCCATAGGAGTGAAGCCTC", "GTGGTCTGCTTCAACATCTGTGGTC",
                    "CTGGACTACAGGTGTTGATAGGAGACG", "GCACAGAGGAACCTTCGGAGAG",
                    "CTCTCAGACTGCAGCTACTGAGC", "GCCGAGAGACATGGCTACATCAAGG",
                    "GGCACCGTCAGGCTGAAGTC"),
       reverseP = c("CTGAGATGATAACACGCTTAGCACCAC", "GAAGGTCTCGAACATGATCTGTGTCATC",
                    "GACAGACTGTTGGTGCTCTGGAC", "GCCAGACACAGCATCTGTCACAC",
                    "CTGCTGATGCCGCTGTATGTGG", "CCTTCATGTCAGGCCACTCATCAG",
                    "CTTCCTCAACCGCCTCTTTGC", "CTGGCTCCAATGATAACAAGGTAGTCG",
                    "CTCAGGTTGCTCTCAGACTCACC", "GATCTATGAGCTTCGGGTAATGGAGAC",
                    "CCACTAACTCTGGCCTCAGTGG", "GTCACGAGGCTTCTGCTGTGAC",
                    "GTCTGTGCTGAGGTTCCAGTCAG", "CAGCTGAGCTTTCTTGCCACAG",
                    "GGCATCGTACTTCTTAGCCAGCTTC"),
       cross_exon_j = c("forward", "reverse", "forward", rep("both", 2),
                        "none", rep("forward", 3), "both", "forward",
                        "both", "forward", "reverse", "both"),
       amplicon = base::paste(c(146, 165, 100, 133, 189, 167, 161, 200,
                                171, 154, 159, 180, 179, 186, 177), "bp")) %>%
  filter(target != "ACTB", target != "TBP") %>% 
  arrange(target) %>% 
  flextable(., col_keys = c("gene_id", "target", "forwardP", "reverseP",
                            "amplicon")) %>%
  theme_box() %>%
  set_header_labels(target = "Target Gene", forwardP = "Forward Primer",
                    reverseP = "Reverse Primer", amplicon = "Amplicon Size",
                    gene_id = "Entrez ID") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::fontsize(., size = 8, part = "all") %>%
  italic(., j = "target") %>%
  flextable::width(width = 0.6) %>%
  flextable::width(j = c("forwardP", "reverseP"), width = 2.2) %>%
  flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = FALSE),
                       opts_word = list(split = F)) %>% 
  footnote(j = 2, i = 7,
           value = as_paragraph(c("Control (house-keeping) gene")),
           ref_symbols = c("HK"), part = "body") %>%
  footnote(j = c(rep("forwardP", 11), rep("reverseP", 6)),
           i = c(1:5, 7:10, 12, 13, 1, 2, 4, 10, 11, 12),
           value = as_paragraph(c("Primer spans exon-exon junction")),
           ref_symbols = c("ExJ"), part = "body", inline = T) %>%
  set_caption("Table S1: Primers for RT-qPCR Validation of RNA-seq data")

## Run gel after qPCR reaction showing single bands to prove specificity of primers
```
**Gene symbols:** glyceraldehyde-3-phosphate dehydrogenase (*GAPDH*); apoptotic peptidase activating factor 1 (*APAF1*); Bcl2 modifying factor (*BMF*); FAS-associated protein with death domain (*FADD*); programmed cell death 4 (*PDCD4*); MAP kinase activating death domain (*MADD*); valosin containing protein (*VCP/p97*); Ubiquitin Recognition Factor in ER Associated Degradation 1 (*UFD1*);  ER degradation enhancing alpha-mannosidase like protein 1 (*EDEM1*); eukaryotic translation initiation factor 3 subunit D and M (*EIF3D* and, *EIF3M*); ribosomal protein L8 and L10a (*RPL8*, and *RPL10A*).

```{r KEGG_result_grofiler2}

gprofiler_kegg <- kegg_res_all %>%
  mutate(regulation = str_replace(query, "^(down|up)_degIDs_\\d{2}hrs$", "\\1"),
         timepoint = str_replace(query, "^(down|up)_degIDs_(\\d{2})hrs$", "\\2-hpi")) %>%
  select(timepoint, regulation, term_name, deg_count = intersection_size, p_value) %>%
  mutate(p_value = format(p_value, digits = 3))
  

gprofiler_kegg %>% flextable(.) %>%
  theme_box() %>% 
  width(., j = "term_name", width = 3) %>%
  set_header_labels(timepoint = "Time Point", regulation = "Regulation",
                    term_name = "KEGG Term", deg_count = "DEG Count",
                      p_value = "P-value (Adjusted)") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::fontsize(part = "all", size = 8) %>%
  set_caption("Table S2: Significantly Enriched KEGG Pathways from DEGs identified at 12- and 24-hpi (Results from the gprofiler2 R package)")
```
\newpage

### Author Contributions   
Conceptualization, Abraham Quaye, Brian D. Poole,  Brett E. Pickett, Bradford K. Berges, and Joel S. Griffitts; methodology, Abraham Quaye; software, Abraham Quaye; validation, Abraham Quaye; formal analysis, Abraham Quaye, Brett E. Pickett; investigation, Abraham Quaye; resources, Brian D. Poole; data curation, Abraham Quaye; writing—original draft preparation, Abraham Quaye; writing—review and editing, Abraham Quaye, Brian D. Poole, Brett E. Pickett, Bradford K. Berges, and Joel S. Griffitts; visualization, Abraham Quaye; 
supervision, Brian D. Poole; project administration, Brian D. Poole; funding acquisition, Brian D. Poole. All authors have read and agreed to the published version of the manuscript.

### Funding  
This research received no external funding.

### Data Availability
 The raw sequencing read data (FastQ), gene expression counts, and total DEGs identified at 12- and 24-hpi have been deposited at the NCBI Gene Expression Omnibus (<http://www.ncbi.nlm.nih.gov/geo>) under accession number GSE286211.

All the code/scripts in the entire analysis pipeline are available on GitHub (<https://github.com/Abraham-Quaye/host_rna_seq>)

### Acknowledgements   
We thank the Office of Research Computing at Brigham Young University for granting us access to the high-performance computing systems to perform the memory-intensive steps in the analysis pipeline of this work.

### Conflicts of Interest   
The authors declare no conflicts of interest.
\newpage

### Abbreviations  
The following are abbreviations used in this manuscript:
```{r list_abbs}
set_flextable_defaults(font.size = 10)
tribble(~Abbreviation, ~Definition,
        "THEV", "Turkey Hemorrhagic Enteritis Virus",
        "HE", "Hemorrhagic Enteritis",
        "hpi", "Hours Post-infection",
        "RT-qPCR", "Reverse Transcriptase Quantitative Polymerase Chain Reaction",
        "FPKM", "Fragments Per Kilobase of transcript per Million mapped reads",
        "ORF", "Open Reading Frame",
        "GCN", "Genome Copy Number",
        "DEG", "Differentially Expressed Gene",
        "GO", "Gene Ontology",
        "KEGG", "Kyoto Encyclopedia of Genes and Genomes",
        "RNA-seq", "RNA sequencing",
        "IMS", "Immunosuppression",
        "VAS", "Virginia Avirulent Strain",
        "DAVID", "Database for Annotation, Visualization and Integrated Discovery",
        "ER", "Endoplasmic Reticulum",
        "UPR", "Unfolded Protein Response",
        "ERAD", "Endoplasmic Reticulum-associated Degradation") %>%
  arrange(Abbreviation) %>% 
  flextable::flextable() %>%
  flextable::theme_box() %>%
  flextable::width(width = 1, j = 1) %>%
  flextable::width(width = 2, j = 2) %>%
  flextable::align(align = "center", part = "all")
```
\newpage

## REFERENCES   
```{=tex}
\setlength{\parindent}{-0.25in}
\setlength{\leftskip}{0.25in}
```
\noindent

::: {#refs}
:::

```{=tex}
\setlength{\parindent}{0in}
\setlength{\leftskip}{0in}
\newpage
```